% ****** Start of file aipsamp.tex ******
%
%   This file is part of the AIP files in the AIP distribution for REVTeX 4.
%   Version 4.1 of REVTeX, October 2009
%
%   Copyright (c) 2009 American Institute of Physics.
%
%   See the AIP README file for restrictions and more information.
%
% TeX'ing this file requires that you have AMS-LaTeX 2.0 installed
% as well as the rest of the prerequisites for REVTeX 4.1
% 
% It also requires running BibTeX. The commands are as follows:
%
%  1)  latex  aipsamp
%  2)  bibtex aipsamp
%  3)  latex  aipsamp
%  4)  latex  aipsamp
%
% Use this file as a source of example code for your aip document.
% Use the file aiptemplate.tex as a template for your document.
\documentclass[%
 aip,
% jmp,
% bmf,
% sd,
% rsi,
 amsmath,amssymb,
%preprint,%
 preprint,%
%author-year,%
%author-numerical,%
% Conference Proceedings
]{revtex4-1}

\usepackage{graphicx}% Include figure files
\usepackage{dcolumn}% Align table columns on decimal point
\usepackage{bm}% bold math
%\usepackage[mathlines]{lineno}% Enable numbering of text and display math
%\linenumbers\relax % Commence numbering lines
\usepackage{hyperref}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{mathptmx}
\usepackage[version=4]{mhchem}
\usepackage{siunitx}
\usepackage{xr}
\externaldocument{SI}
\DeclareSIUnit[number-unit-product = {}]
\molar{\,M}
\DeclareSIUnit[number-unit-product = {}]
\ppm{\,ppm}

%%%%LaTeX Aliases %%%%%%%%%%
\newcommand{\ca}{\ce{C_\alpha} }
\newcommand{\cb}{\ce{C_\beta} }

\makeatletter
\newcommand*{\addFileDependency}[1]{% argument=file name and extension
  \typeout{(#1)}
  \@addtofilelist{#1}
  \IfFileExists{#1}{}{\typeout{No file #1.}}
}
\makeatother

\newcommand*{\myexternaldocument}[1]{%
    \externaldocument{#1}%
    \addFileDependency{#1.tex}%
    \addFileDependency{#1.aux}%
}

\myexternaldocument{SI}


\begin{document}

%\preprint{AIP/123-QED}

\title[The dominant activated state of KcsA]{Informing NMR experiments with molecular dynamics simulations to characterize the dominant activated state of the KcsA ion channel}
% Force line breaks with \\

\author{Sergio P{\'e}rez-Conesa}
\thanks{These three authors contributed equally}
 \affiliation{KTH Royal Institute of Technology, Science for Life Laboratory, Stockholm, Sweden}%

\author{Eric G. Keeler}
\thanks{These three authors contributed equally}
\affiliation{Department of Chemistry, Columbia University, New York, New York 10027, United States}

\author{Dongyu Zhang}
\thanks{These three authors contributed equally}
\affiliation{Department of Chemistry, Columbia University, New York, New York 10027, United States}
\affiliation{Current Address: Plexxikon Inc., Berkeley, CA 94710, United States}

\author{Lucie Delemotte}
\thanks{Correspondance to: lucied@kth.se, aem5@columbia.edu }
\affiliation{KTH Royal Institute of Technology, Science for Life Laboratory, Stockholm, Sweden}

\author{Ann E. McDermott}
\thanks{Correspondance to: lucied@kth.se, aem5@columbia.edu }
\affiliation{Department of Chemistry, Columbia University, New York, New York 10027, United States}

\date{\today}% It is always \today, today,
             %  but any date may be explicitly specified

\begin{abstract}
As the first potassium channel with a X-ray structure determined, and given its homology to eukaryotic channels, the pH-gated prokaryotic channel KcsA has been extensively studied. Nevertheless, questions related in particular to the allosteric coupling between its gates remain open. The many currently available X-ray crystallography structures appear to correspond to various stages of activation and inactivation, offering insights into the molecular basis of these mechanisms. Since these studies have required mutations, complexation with antibodies, and substitution of detergents in place of lipids, examining the channel under more native conditions is desirable. Solid-state NMR (SSNMR) can be used to study the wild-type protein under activating conditions (low pH), at room temperature, and in bacteriomimetic liposomes. 
In this work, we sought to structurally assign the activated state present in SSNMR experiments. We used a combination of molecular dynamics (MD) simulations, chemical shift prediction algorithms, and Bayesian inference techniques to determine which of the most plausible X-ray structures resolved to date best represents the activated state captured in SSNMR. We first identified specific nuclei with simulated NMR chemical shifts that differed significantly when comparing partially open vs. fully open ensembles from MD simulations. The simulated NMR chemical shifts for those specific nuclei were then compared to experimental ones, revealing that the simulation of the partially open state was in good agreement with the SSNMR data. Nuclei that discriminate effectively between partially and fully open states belong to residues spread over the sequence and provide a molecular level description of the conformational change.
\end{abstract}

\maketitle

\section{Introduction}
Transmembrane allostery is at the heart of many signalling events in human biochemistry.\cite{Cournia2020} KcsA has proved to be an excellent model system for studying this intriguing phenomenon, and has also served as a prototype for ion channel function.\cite{Xu2017} Activation coupled inactivation, a hallmark functional feature of essentially all ion channels, has been studied in KcsA by numerous biophysical and biochemical tools \cite{Li2018,Cuello2017,Xu2019,Furini2020,Ostmeyer2013,Sun2020,Ader2009,Labro2018,Delemotte2018}. The literature in aggregate has resulted in a dominant hypothesis in which manipulation of an intracellular gate results in opening to a metastable activated state that lives for some milliseconds, during which time it directly triggers a slow allosteric inactivation involving loss of ions in the selectivity filter \cite{Li2018,Xu2019,Zhang2020a}. The details of the opening and coupled inactivation processes remain unclear and of great interest. Identifying the stable intermediates of these processes is central to clarifying the molecular basis for inactivation coupling in channels. 

Solid-state nuclear magnetic resonance (SSNMR) studies of KcsA have provided unique insights regarding dynamics and thermodynamics of full length wild-type channels in hydrated lipid bilayers. These studies have the advantage of being carried out in near-native conditions, without use of antibodies, and with freely varied ligand concentrations ([\ce{K+}] and [\ce{H+}]). In our NMR studies, we have stabilized three states, which we have referred to as deactivated (stabilized at neutral pH and high [\ce{K+}] ), activated (stabilized at low pH and high [\ce{K+}] ) and inactivated (stabilized at low [\ce{K+}]), with a naming convention selected because of the functional consequences of varying pH and [\ce{K+}] (Fig. \ref{general_scheme}.a). An important prerequisite for interpretation of the NMR data is identifying (for each of these experimental conditions) the atomic arrangement that best represents the majority species in the sample. Identification of the deactivated and inactivated states is largely straightforward given the sample conditions used and the stability of these states: both X-ray diffraction (XRD) and solid-state NMR data suggest that the samples are homogeneous and robustly reproducible. Here the deactivated state corresponds to a channel with a closed activation gate (also called hydrophobic bundle crossing or inner helix bundle or inner gate) and a conductive selectivity filter fully loaded with potassium ions (PDB ID 1K4C)\cite{Zhou2001,Bhate2013,Wylie2014,Medeiros-Silva2016,Jekhmane2019}. The inactivated state corresponds to a channel with a wide open inner gate and a pinched selectivity filter depleted of ions (PDB ID 5VKE, 5VKH, 3F7V, 3F7Y, 3F5W)\cite{Cuello2010,Ader2009,Bhate2010,Cuello2017} (Fig. \ref{general_scheme}.a). 

The activated state, the only state that conducts ions, on the other hand, presents a particular challenge. Indeed, the seminal crystallography work by Cuello et al. showed that the development of a wide open activation gate was coupled to the appearance an inactive selectivity filter, and many other studies showed that the selectivity filter tended to collapse when the gate was held open, an observation compatible with the metastable short-lived nature of the activated state.\cite{Cuello2010,Li2018,Furini2020} Although the activated state has proven difficult to stabilize, X-ray crystallography studies involving a number of mutations and antibody binding succeeded in stabilizing a family of configurations with a conductive selectivity filter and the activation gate assuming a range of opening degrees, suggesting an almost continuous opening process\cite{Cuello2010}. A mutant channel (where all but one of the proton sensor charges were neutralized) was captured together with a conductive selectivity filter/Partially Open (PO) pore, with degrees of opening varying between \SI{14}{\angstrom} and \SI{16}{\angstrom} cross bundle distances at the narrowest site in the pore (measured as the distance between T112 \ca of opposing subunits) (PDB IDs 3FB5 3FB6)\cite{Cuello2010}. Recently a non-inactivating mutant E71A structure was engineered with cysteines at positions 28 and 118 forming disulfide bonds to capture the channel in a Fully Open (FO) state, revealing a wider pore, of \SI{23}{\angstrom} cross bundle distance at the inner gate (PDB ID 5VK6). In this structure, interestingly, the gate was as wide open as in the inactivated state structure\cite{Cuello2017}.
 
These results raise a question about the degree of opening in the low pH near-native condition SSNMR activated samples. \textit{De novo} structure determination of such a system from NMR restraints has not yet been reported. In this work, we have used an alternative approach to compare the compatibility of configurational ensembles obtained using MD simulations initiated from XRD structures with the NMR spectra. Chemical shifts (CS) have been shown to be very sensitive to biopolymer conformation, and in some cases contain sufficient information to correctly identify 3-dimensional structures.\cite{DeDios1993,Oldfield1995,Wishart2002,Shen2008,Cavalli2007,Niklasson2015,Berjanskii2015} The use of chemical shift prediction tools has also been extended to restrain MD simulations in order to determine ensemble configurations compatible with the experiment\cite{camilloni2012characterization} and even to account for averaging of the experimental chemical shifts due to internal dynamics.\cite{Robustelli2012,Markwick2010,Li2010,Li2012,Li2015} Therefore we leverage CS information to identify the most likely structure for the NMR activated state. To do so, we ran MD simulations based on the two most likely representative XRD structures of the activated state (Partially Open PO, 3FB5 and Fully Open FO, 5VK6) and the deactivated state (Closed C, 5VKH) and predicted the ensemble of CS sampled by the trajectories of the NMR-assigned nuclei in the different residues. We then calculated relative chemical shifts ($\Delta \text{CS}_\text{sim}^\text{X} = \text{CS}_\text{sim}^\text{X} - \text{CS}_\text{sim}^\text{Closed}$, where X represents the FO or PO state), which are expected to be less affected by systematic errors than absolute chemical shifts. Using Bayesian inference, we determined signals that distinguished the PO and FO ensembles. We then calculated the deviation of these 49 $\Delta \text{CS}_\text{sim}^\text{X}$ from the $\Delta \text{CS}_\text{exp}$ derived from the SSNMR experiments, leading us to conclude that the activated state probed in the SSNMR is closer to the PO than the FO state. 

\begin{figure}
\includegraphics[width=\textwidth]{Fig1.pdf}% Here is how to import EPS art
\caption{\label{general_scheme} \scriptsize
Summary scheme of the various computational steps carried out in this work. (a) Illustration of the gating cycle of KcsA and the corresponding X-ray crystallography structures with the nomenclature used for the different states. (b) Bayes law (top equation) and its terms: the posterior distribution ( $p( \mu, \sigma, \alpha |\, d)$ ) is proportional to the product of the likelihood ( $p( d |\, \mu, \sigma, \alpha )$ ) and the priors ( $p( \mu, \sigma, \alpha )$ ) where $\mu$, $\sigma$, $\alpha$ are the position, scale, and skewness parameters of a skew-normal distribution and $d$ is the chemical shift data for a particular nucleus of the protein. The posterior probability of the parameters of a skew-normal distribution representative of the CS calculated from the MD simulation ensemble is proportional to the product of the likelihood of the data (predicted chemical shifts) multiplied by the prior distribution of the parameters (chosen here as uninformative). The definition of the mean of this skew-normal distribution  is reported as the $\text{CS}_{\text{sim}}^{\text{X}}$ with its 94\% credible interval. (c) Criteria imposed to consider a given residue's CS as statistically significant. This is illustrated using two idealized CS-likelihoods obtained from the posteriors inferred for the two possible activated states, showing how their overlap and location affects statistical significance. This was done by assessing the difference in means between the two states ($\text{CS}_{\text{sim}}^{\text{FO}}-\text{CS}_{\text{sim}}^{\text{PO}}$) and the effect size of this difference $\big(\frac{\text{CS}_{\text{sim}}^{\text{FO}}-\text{CS}_{\text{sim}}^{\text{PO}}}{\text{std}_{pool}}\big)$ where $\text{std}_{pool}=\sqrt{\frac{var(\text{CS}^{\text{FO}}_{\text{sim}})-var(\text{CS}^{\text{PO}}_{\text{sim}})}{2}}$ and $var$ is the variance function. (d) Assignment of nuclei indicating compatibility of the activated state NMR sample with the PO or FO MD simulation ensembles. The method we use involves calculating the CS difference between the Closed (C) and the Partially Open / Fully Open (PO/FO) states (labeled X) measured using SSNMR ($\Delta \text{CS}_\text{exp}$) and predicted from the MD simulations ensemble ($\Delta \text{CS}_\text{sim}^\text{X}$) for the nuclei determined in (c). If the difference between the $\Delta \text{CS}$ measured experimentally and using the simulation ensemble ($\Delta\Delta \text{CS}^\text{X}$) is smaller for the PO than the FO state, this nucleus supports the conclusions that the experimentally observed activated state is the PO state seen in crystal structures. The reverse is true if the difference is smaller for the FO state.
}
\end{figure}

\section{Methods}

\subsection{Expression and purification of U-\ce{^13C},\ce{^15N} wild-type KcsA}
Uniformly \ce{^13C},\ce{^15N} labeled KcsA was overexpressed and purified as described previously from \textit{Escherichia coli} PASK90/JM83 and PASK90/BL21(DE3) cells in M9 minimal media.\cite{Bhate2013,Xu2017,Zhang2020a} Purified, uniformly \ce{^13C}-\ce{^15N} labeled, KcsA in detergent dodecyl $\beta$-D-maltoside (DM) (Anatrace, Maumee, OH) was reconstituted into liposomes by reducing the detergent concentration through dialysis; the liposomes consisted of 3:1 1,2-dioleoyl-sn-glycero-3-phosphoethanolamine (DOPE) : 1,2-di-(9Z-octadecenoyl)-sn-glycero-3-phospho-(10-rac-glycerol) (DOPG) (Avanti Polar Lipids, Alabaster, AL) at pH 4.0 and \SI{50}{\milli\molar} \ce{KCl} with a lipid to protein weight ratio of 1:1. The total ionic strength was kept at \SI{100}{\milli\molar} by compensation with \ce{NaCl}.

\subsection{NMR Spectroscopy}
25 (or 9) \si{\milli\gram} of hydrated proteoliposome sample was centrifuged into a Bruker 3.2 (or 1.9) \si{\milli\meter} rotor after 3-5 freeze-thaw cycles. SSNMR experiments were performed on either a Bruker AVANCE II or NEO spectrometer operating at \SI{21.1}{\tesla} ($\omega_{0,\text{H}}/2\pi$ = \SI{900}{\mega\hertz}) or \SI{17.6}{\tesla} ($\omega_{0,\text{H}}/2\pi$ = \SI{750}{\mega\hertz}). Experiments were collected with either a \SI{3.2}{\milli\meter} HCN E-free probe (\SI{900}{\mega\hertz}) or a \SI{1.9}{\milli\meter} HCN probe (\SI{750}{\mega\hertz}) (Bruker BioSpin, Billerica, MA), as indicated. 

Experiments at \SI{900}{\mega\hertz} were collected with a spinning frequency, $\omega_{r}/2\pi$, of $16666\pm\SI{10}{\hertz}$ and a set point temperature of $267\pm\SI{1}{\kelvin}$. The sample temperature is estimated to be $290\pm \SI{5}{\kelvin}$ due to sample heating from MAS and RF pulses.\cite{Zhang2019} Standard $\pi/2$ pulse lengths of \SI{2.55}{\micro\second}, \SI{4}{\micro\second}, and \SI{5.1}{\micro\second} were used for the \ce{^1H}, \ce{^13C}, and \ce{^15N} channels, respectively, corresponding to $\omega_1/2\pi$ = \SI{98}{\kilo\hertz} (1H), \SI{62.5}{\kilo\hertz} (\ce{^13C}), and \SI{49}{\kilo\hertz} (\ce{^15N}). Two-dimensional (2D) \ce{^13C}-\ce{^13C}, 2D \ce{^13C}-\ce{^15N} (NcaCX, NcoCX), and three-dimensional (3D) \ce{^13C}-\ce{^13C}-\ce{^15N} (NCOCX, NCACX) correlation experiments were used to determine the chemical shift assignments.\cite{Pauli2001,Igumenova2004,Li2007} The \ce{^13C}-\ce{^13C} mixing was performed using dipolar assisted rotational resonance (DARR) mixing of 50 ms.\cite{Takegoshi2001} The heteronuclear polarization transfer used SPECtrally Induced Filtering In Combination with Cross Polarization (SPECIFIC-CP).\cite{Baldus1998} SPECIFIC-CP standard RF conditions of $\omega_{1,\text{C}}$ = 1.5$\omega_{r}$ and $\omega_{1,\text{N}}$ = 2.5$\omega_{r}$ for NCA, and $\omega_{1,\text{C}}$ = 2.5$\omega_{r}$ and $\omega_{1,\text{N}}$ = 1.5$\omega_{r}$ for NCO and contact times between 4.5 and \SI{5.0}{\milli\second} were used for SPECIFIC-CP experiments. \ce{^1H} decoupling of $\omega_{1,\text{H}}/2\pi$ > \SI{90}{\kilo\hertz} during acquisition was performed using SPINAL-64 decoupling.\cite{Fung2000} More detailed experimental parameters for the 3D experiments are given in Table \ref{SI_tb_900exp}. 

Experiments at \SI{750}{\mega\hertz} were collected with spinning frequencies of $16000\pm10$ or $33333\pm\SI{10}{ \hertz}$ and a set point temperature of 230 ($\omega_{r}/2\pi$ = \SI{33333}{\hertz}) or 240 ($\omega_{r}/2\pi$ = \SI{16000}{\hertz}) $\pm$ \SI{1}{\kelvin}. The sample temperature is estimated to be 290 $\pm$ \SI{5}{\kelvin} due to sample heating from MAS and RF pulses.\cite{Zhang2019} Standard $\pi/2$ pulse lengths of \SI{2.55}{\micro\second}, \SI{3.3}{\micro\second}, and \SI{4.3}{\micro\second} were used for the \ce{^1H}, \ce{^13C}, and \ce{^15N} channels, respectively, corresponding to $\omega_1/2\pi$ = \SI{98}{\kilo\hertz} (\ce{^1H}), \SI{76}{\kilo\hertz}(\ce{^13C}), and \SI{58}{\kilo\hertz} (\ce{^15N}). The same 3D experiments for chemical shift assignments were performed at \SI{750}{\mega\hertz}. The \ce{^13C}-\ce{^13C} mixing at \SI{750}{\mega\hertz} was performed using COmbined $R2^v_n$-Driven (CORD) mixing of \SI{50}{\milli\second}.\cite{Hou2013} SPECIFIC-CP RF conditions of $\omega_{1,\text{C}}$ = 0.5$\omega_{r}$ and $\omega_{1,\text{N}}$ = $ 1.5\omega_{r}$ for NCA and NCO at $\omega_{r}/2\pi$ = \SI{33333}{\hertz} and $\omega_{1,\text{C}}$ = $1.5\omega_{r}$ and $\omega_{1,\text{N}}$ = $2.5\omega_{r}$ for NCA, and $\omega_{1,\text{C}}$ = $0.5\omega_{r}$ and $\omega_{1,\text{N}}$ = $1.5\omega_{r}$ for NCO at $\omega_{r}/2\pi$ = \SI{16000 }{\hertz}. SPECIFIC-CP contact times of between 4.5 and \SI{5.0}{\milli\second} were used. A 2D \ce{^13C}-\ce{^15N} ZF-TEDOR experiment\cite{Jaroniec2002} was collected with \SI{1.5}{\milli\second} mixing and $\omega_{r}/2\pi$ = 16000 Hz for identification of the proline chemical shifts. \ce{^1H} decoupling of $\omega_{1,\text{H}}/2\pi$ > \SI{90}{\kilo\hertz} during acquisition was performed using swept-frequency two-pulse phase-modulated (SW$_f$-TPPM) decoupling.\cite{Thakur2006} More detailed experimental parameters for the 3D experiments are given in Table \ref{SI_tb_750exp}.

All data were processed in Topspin (Bruker BioSpin) or NMRPipe.\cite{Delaglio1995} The \ce{^13C} and \ce{^15N} chemical shifts were referenced via the substitution method to the methylene peak of adamantane at \SI{40.48}{\ppm} (0.5\% DSS in \ce{D2O} scale), and to the peak of \ce{^15NH4Cl}  to \SI{39.3 }{\ppm} (at \SI{298}{\kelvin}, \ce{NH3} (liquid) scale), respectively.\cite{Morcombe2003,McDermott2007,Harris2008,Bertani2014} Peak picking and assignments were made in CcpNmr.\cite{Vranken2005} All the spin systems were identified first in each 3D spectrum and then the residue type and their secondary structure were predicted using PLUQin.\cite{Fritzsching2016} Backbone resonance assignments were determined based on correlations in the 3D NCACX, NCOCX, and CANCO spectra, beginning from unique spin systems (for which the residue type was identified using PLUQin). Sidechain residue assignments were determined mainly based on 3D NCACX, NCOCX, and 2D \ce{^13C}-\ce{^13C} DARR experiments. The following tolerance thresholds were applied for the agreement of chemical shifts when comparing the various spectra of the activated state for assignments: \SI{0.2}{\ppm} for \ce{^13C}, and \SI{0.5}{\ppm} for \ce{^15N}. Monte Carlo/simulated annealing (MC/SA) algorithms in the software packages NSGA-II and MC/SA were used to facilitate assignments.\cite{Yang2013}

\subsection{Molecular dynamics simulations}
Coordinates for PO, FO, and C KcsA structures (PDB IDs: 3FB5, 5VK6, 5VKH)\cite{Cuello2010,Cuello2017} were prepared using CHARMM-GUI\cite{Jo2017}. The constructs included residues 26-121. For 3FB5, Rosetta\cite{DiMaio2011} was used to add missing residues 114-121 in $\alpha$-helical conformation. The N- and C- termini were capped by methylation (NME) and acetylation (ACE). We note that the pH sensor was not included in the simulations to keep consistent the chain-lengths across states. Using PYMOL\cite{PyMOL} the following mutations were reverted: for the FO state (5VK6), A28C, E71A, L90C, R117Q, E118C, E120Q, R121Q; for the PO state (3FB5), L90C, R117Q, E120Q, R121Q and for the closed state (5VKH), Y82A, L90C, F103A. The sidechain of residue E71 was oriented to form a hydrogen-bond with D80 as done in previous works in the literature\cite{Li2018}. None of the residues that were mutated in the activated states (3FB5 and 5VK6) were later found to be important to discriminate between the PO and FO states. For all states, residues E71, E118, and E120 were protonated since H-NMR spectra showed these residues to be protonated in the activated state.\cite{Bhate2012,Wylie2014} We note that the simulated protein contains no histidines due to the truncation of the sequence. The selectivity filter was initially fully loaded with \ce{K+} in agreement with the X-ray diffraction and crystallization waters bound to the channel were included. The protein was embedded in a DOPE:DOPG (3:1) membrane bilayer containing 150 lipids and surrounded by a neutralizing solution of 50 mM NaCl and 50 mM KCl at a ratio of 75 TIP3P\cite{TIP3P_JChemPhys_Jorgensen_1983} rigid water-molecules for each lipid molecule. All states were modelled including the co-purifying DOPG lipids, as identified in the X-ray diffraction structures. 

The systems were equilibrated using the CHARMM36 force field\cite{Best2012} in an extended CHARMM-GUI protocol. \SI{1}{\micro\second}-long simulations were performed using GROMACS2019\cite{GROMACS_Abraham2015}. The Parrinello-Rahman barostat\cite{Parrinello1981} ( $P=\SI{1}{\bar}$ $\tau = \SI{5}{\pico\second}$  ) and Bussi thermostat\cite{Bussi2007} ( $T=\SI{290}{\kelvin}$ $\tau = \SI{1}{\pico\second}$  ) were employed to mimic experimental conditions. The parts of the simulation after a water molecule enters the selectivity filter were excluded since such is a symptom of inactivation (see Fig \ref{SI_SF_occupation})\cite{Furini2020}. Additionally, the initial parts of the simulation where the inner-gate distance was unstable were also excluded from analysis (see Fig \ref{SI_IG}). Additional technical details of the simulation can be obtained from the simulation files and trajectories freely available at \url{https://github.com/delemottelab/Informing_NMR_experiments_w_MD} and \url{https://osf.io/6h2z5/}. 

\subsection{Statistical Inference of Chemical Shifts from MD simulations}

N, C, \ca, and \cb CS of the simulation snapshots were calculated using the SPARTA+\cite{Shen2010} and SHIFTX2\cite{Han2011} chemical shift prediction tools, with their default options. For SHIFTX2, this implies turning on using the homology-based algorithm SHIFTY+ if possible. Nevertheless, due to a lack of homology with the sequences in RefDB\cite{Zhang2003}, this selection had no effect. Although quantum mechanical NMR chemical shift prediction tools are more accurate than empirical methods, using them would constitute an overly ambitious understaking due to the size of the protein and the number of MD snapshots in the configurational ensemble. Only nuclei experimentally assigned for both states were considered. Since KcsA is a homotetramer, we were able to collect four times the data per residue under the assumption that the trajectories of the monomers were independent. Nevertheless, since there are contact surfaces between the monomers that can affect the CS prediction, the prediction software was given the full tetramer and the data obtained for the four equivalent subunits was concatenated \textit{a posteriori}. The CS-prediction error for each MD simulation snapshot that is associated to the accuracy of the prediction tool was not modelled. The overall agreement of the conclusions drawn for both methods highlights the general robustness of the methods. 

The statistical distributions of the CS were modelled using Bayesian inference with the python libraries PYMC3\cite{Salvatier2016} and Arviz\cite{arviz_2019}. In line with this work, there have been previous studies that used Bayesian inference methods to analyze heterogeneous ensembles of CS derived from chemical shift predictions or calculations.\cite{Bonomi2016,Bratholm2017,Bratholm2015} Bayesian inference assumes that the probability distribution of a random variable has a given functional form and enables to infer the probability of the parameters of the distribution given the data sampled ($d$). A schematic description of the procedure is found in Fig. \ref{general_scheme}b . The CS of each particular nucleus, state, residue, and prediction method was considered as an independent variable and was therefore modelled separately in a non-hierarchical framework. For every variable, two thousand data points were selected randomly out of all the time frames obtained from each MD trajectory. 

The likelihood function chosen was a skew-normal distribution as implemented by PMC3:
\begin{equation}
L(d|\,\mu, \sigma, \alpha)= \frac{2}{\sigma}\cdot
\phi\left(\frac{d-\mu}{\sigma}\right)\cdot
\Phi\left(\left(d-\mu\right)\frac{\alpha}{\sigma}\right)\cdot
\end{equation}
where $d$ is the data sampled for the CS$_\text{sim}^\text{X}$ of a particular nucleus, and for a particular residue in that particular state X. $\phi$ and $\Phi$ are the probability distribution function and cumulative distribution function of the Gaussian distribution:
\begin{equation}
\phi(x)=\frac{1}{\sqrt{2}}e^{-\frac{1}{2}x^2}
\end{equation}
\begin{equation}
\Phi(x) =\frac{1}{2}\left[1+
\frac{2}{\sqrt{\pi}}\int^{x/\sqrt{2}}_{0}t^2dt
\right]
\end{equation}
The skew-normal function has three parameters: the location ($\mu$), the scale parameter ($\sigma$), and skewness parameter ($\alpha$). In order to conduct Bayesian inference, previous assumptions of the parameters are included by assigning probability distribution functions, \textit{priors}, to them. For the $\mu$ parameters, normal distributions are assigned:
\begin{equation}
    N(\mu)=\frac{1}{\SI{10}{\ppm}\sqrt{2}}e^{-\frac{1}{2}\left(
    \frac{\mu-\bar{d}}{\SI{10}{\ppm}}
    \right)^2}
\end{equation}

Where $\bar{d}$ is the average value of the sampled CS$_\text{sim}^\text{X}$ data (d) and \SI{10}{\ppm} the scale of the normal distribution. 
For $\sigma$, the priors assigned are half-Cauchy distributions, assigned with a beta parameter of \SI{10}{\ppm}:
\begin{equation}
    C(\sigma)=\frac{1}
    {\pi\SI{10}{\ppm}\left[ 
    1+\left(
    \frac{\sigma}{\SI{10}{\ppm}}
    \right)^2\right]
    }
    \forall \sigma \geq 0
\end{equation}
For $\alpha$, normal distributions are also assigned:
\begin{equation}
    N(\alpha)=\frac{1}{\SI{10}{\ppm}\sqrt{2}}e^{-\frac{1}{2}\left(
    \frac{\alpha}{\SI{10}{\ppm}}
    \right)^2}
\end{equation}
The posterior distribution was sampled using the NUTS\cite{sanjeevi2017nuts} algorithm with 4 chains. It is important to remark that the mean of a skew-normal distribution is not the location parameter $\mu$ but rather $mean = \mu + \sigma\sqrt{\frac{2}{\pi}}\frac{\alpha}{\sqrt{1+\alpha^2}}$ . This mean value of the distribution is what we will consider the chemical shift $\text{CS}_{\text{sim}}^{\text{X}}$ of state X predicted by our MD simulation, or in other words the simulated position of the peak in the NMR spectrum. This corresponds to a physical assumption of rapid population weighted average of the chemical shifts sampled in the protein dynamics. In similar fashion, the distribution variance was calculated as $var = \sigma^2\left(1-\frac{2\alpha^2}{\left(\alpha^2+1\right)\pi}\right)$. Posterior predictive checks were carried out after the inference to test the adequacy of the model (Fig \ref{SI_ppc}). The main features of the data distributions are generally in good agreement with the model. Even in the case where the distribution is bimodal, the envelope of the skew-normal function covers the bimodal distribution (Fig \ref{SI_ppc}).  Additional tests and inference information can be found in the SI and \url{https://github.com/delemottelab/Informing_NMR_experiments_w_MD}. 

All quantities obtained with Bayesian inference are not assigned a single value, but rather a probability distribution. As a consequence, for simplicity, the distributions of these random variables can be summarized by the interval that contains 94\% of its probability. This interval is known as the 94\% credible interval (CI) and is written as its center plus or minus the distance to the bounds of the interval. A graphical representation can be found in Fig \ref{general_scheme}b. 

\subsection{Selecting nuclei whose CS differ statistically between states}

The next step was to determine which nuclei were statistically different in the PO and FO trajectories. For this, the difference between the simulated chemical shifts for both states modeled as described above was calculated ($\text{CS}_{\text{sim}}^{\text{FO}}-\text{CS}_{\text{sim}}^{\text{PO}}$), as well as the effect size of this difference $\big(\frac{\text{CS}_{\text{sim}}^{\text{FO}}-\text{CS}_{\text{sim}}^{\text{PO}}}{\text{std}_{pool}}\big)$ where $\text{std}_{pool}=\sqrt{\frac{var(\text{CS}^{\text{FO}}_{\text{sim}})-var(\text{CS}^{\text{PO}}_{\text{sim}})}{2}}$ and $var$ is the variance function. The first expression measures if there is a difference between the CS predicted for both states and the second if this difference is large compared to the variance of the distributions. The importance and graphical interpretation of the difference in means and effect size can be seen in Figure \ref{general_scheme}c. 

 A CS variable for a given nucleus in a given residue generated using a given chemical shift prediction method is considered to be have discrimination power if the center of the difference in means credible interval is greater in absolute value than the experimental tolerance (\SI{0.2}{\ppm} for C, \ca, \cb and \SI{0.5}{\ppm} for N) and the center of the effect size credible interval is larger than 0.5 in absolute value. An effect size greater than 0.5 is considered medium as a general rule in statistics\cite{Sawilowsky2009}. 
This procedure is repeated independently for each studied nucleus, for each chemical shift prediction method, and for each residue. 

\subsection{Comparing CS differences from MD simulations and NMR}
Site specific CS for the PO and FO states were contrasted with experimentally measured CS to assess which state is present in the experiment. A schematic representation of the procedure can be found in Fig. \ref{general_scheme}d. 94\% credible intervals were calculated for both the FO and PO states (designated "X" since it is to be determined which is correct), using the expressions $\Delta \text{CS}_\text{sim}^\text{X}=\text{CS}_\text{sim}^\text{X}-\text{CS}_\text{sim}^\text{C}$, $\Delta \text{CS}_\text{exp}=\text{CS}_\text{exp}^\text{pH=4,act}-\text{CS}_\text{exp}^\text{ph=7.5,deact}$, and $\Delta\Delta \text{CS}^\text{X}=\Delta \text{CS}_\text{sim}^\text{X} - \Delta \text{CS}_\text{exp}$. The state, X, for which $\Delta\Delta \text{CS}^\text{X}$ is lower in magnitude has a predicted $\Delta \text{CS}_\text{sim}^\text{X}$ closer to the experimentally measured $\Delta \text{CS}_\text{exp}$ and we conclude it to be likely that this state is dominant in the experiment at that nucleus. This procedure was repeated for all nuclei in all residues for which experimental shifts were available, using both CS prediction software packages. It is interesting to note than point measures of error such as root mean squared error (RMSE) give differences between the states that are in the range of the experimental uncertainty (Table \ref{SI_tb_RMSE}). These differences only start to become substantial if the statistical filtering information and the closed state is used as reference. 

\section{Results}

\subsection{NMR CS Assignments of the activated state sample}

Chemical shift assignments of the activated state (pH 4.0, \SI{50}{\milli\molar} [\ce{K+}]) of KcsA were determined using three-dimensional spectra (NCACX, NCOCX, and CANCO) acquired at two different external magnetic fields (900 and 750 MHz), see Table \ref{SI_tb_activated_exp_CS}. Representative strip plots of the backbone walk for the assignments of residues T74 to G79 using these spectra are given in Figure \ref{SI_3D_strip_plot}. The completeness of the backbone and sidechain assignments, along with the experimental connectivities of each resonance in each experiment, are presented in a schematic representation in Figure \ref{SI_NMR_assign_scheme}. A number of the assigned resonances, including some that are used in the analysis presented here, are well resolved in various 2D spectra, as illustrated in Figures \ref{fg:NMR_2D_markers} and \ref{SI_NMR_full_2Ds}. Chemical shifts for the deactivated state at pH 7.5, \SI{50}{\milli\molar} [\ce{K+}]  (previously reported,\cite{Wylie2014} Table \ref{SI_tb_deactivated_exp_CS}) were re-referenced using the protocol described in this paper so as to be comparable to the shifts determined here for the activated state (pH 4.0, \SI{50}{\milli\molar} [\ce{K+}]). Distributions of the  chemical shift differences (comparing  activated and deactivated states) indicate that the chemical shift referencing is consistent for the two datasets (Figure \ref{SI_Exp_Hist}). In fact, the site-specific chemical shifts were found to be similar for the activated and deactivated states, with few notable exceptions, suggesting that the two states adopt similar structural folds, as has been indicated from X-ray crystallography studies (Table \ref{SI_tb_DeltaCSexp}). Resonances were considered to be experimental markers if the chemical shift difference (activated vs. deactivated, $\text{CS}_{\text{exp}}^{\text{pH 4,act}} - \text{CS}_{\text{exp}}^{\text{pH 7.5,deact}}$) was greater than the experimental tolerance (0.2 ppm for \ce{^13C} and 0.5 ppm for \ce{^15N}). If this difference does not exceed this threshold the assigned resonance was considered to be an experimental spectator.

\begin{figure}[tbp]
	\includegraphics[width=0.45\textwidth]{KcsA_NCA_markers_v2.eps}
	 \caption{\label{fg:NMR_2D_markers}\scriptsize
    Representative experimental NMR spectra of KcsA in the activated state (3:1 DOPE:DOPG, 50 mM KCl, pH 4.0). (a) NCA region of 2D \ce{^15N}-\ce{^13}C NcaCX spectrum with assigned peaks shown. All peaks in the NcaCX spectrum are C$_{\alpha}^i$-N$^i$ unless noted otherwise. (b) NCO region of 2D \ce{^15N}-\ce{^13C} NcoCX spectrum with assigned peaks shown. All peaks in the NcoCX spectrum are C$^{i-1}$-N$^i$ unless noted otherwise. (c) \ca-\cb region of 2D \ce{^13C}-\ce{^13C} DARR spectrum with assigned peaks labeled. All peaks in the DARR spectrum are C$_{\alpha}^{i}$-C$_{\beta}^{i}$ unless noted otherwise. Peaks used to distinguish between fully open vs partially open (identified by statistical inference analysis of the CS predictions of MD snapshots) are labeled in red, and those identified (by the same methods) as spectators are labeled in gray.}
\end{figure}

\subsection{Sampling the deactivated and activated KcsA conformational ensemble using MD simulations}

In order to run MD simulations to characterize the activated state, the most plausible XRD structures were selected as starting coordinates. There is a rich set of such structures for KcsA, which mostly differ in their inner gate opening (measured as the distance between T112 \ca from opposing subunits): 3FB5 (\SI{14}{\angstrom}), 3FB6 (\SI{16}{\angstrom}), 5VK6 (\SI{22}{\angstrom}) and 3F5W (\SI{31}{\angstrom})\cite{Cuello2010,Cuello2017}. In this work, the 3FB5 structure was used instead of 3FB6 because of its higher resolution and therefore fewer missing side chains. Nevertheless, the two structures are very similar and the MD simulation initiated from 3FB5 coordinates explores  openings compatible with 3FB6 (Fig. \ref{SI_ppc}). 3F5W was found to have an unstable inner-gate that moved towards openings compatible with 5VK6 and therefore was not included in this study (Fig. \ref{SI_IG_3F5W}). Therefore, the initial configurations were prepared based on the 3FB5 PO structure, and from the 5VK6 FO structure. We also ran simulations of the C state (conductive selectivity filter, closed inner gate), to compare the activated state to, preparing a system based on the 5VKH high resolution crystal structure. 

The simulation conditions were chosen to match the experimental conditions as closely as possible. The lipid and solution composition in the computational system were identical to the ones in the NMR experiment: 3:1 DOPE:DOPG in a symmetrical bilayer and a concentration of 50 mM of NaCl and 50 mM of KCl in the solution. The simulation temperature and pressure were set to \SI{290}{\kelvin} and 1 atm, respectively, thus close to experimental conditions. To mimic the pH of the experiments, acidic residues E71, E118, and E120 were protonated and the protonation states were kept fixed due to limitations of classical MD simulations. The main difference between the simulated and experimental constructs is the truncation of the sequence to keep residues 26 to 121 due to the lack of experimental coordinates to model the intracellular domains. In the three states, the models were overall stable over the hundreds of nanoseconds timescale, as characterized by the RMSD (Fig. \ref{SI_RMSD}).

The opening of the inner gate was found to be fairly stable for both the C and FO states (Fig \ref{SI_ppc}). In contrast, the PO state inner gate experienced some initial instability, followed by asymmetrization, and finally a drift towards closing. For this reason data collected between \SI{400}{\nano\second} and \SI{1000}{\nano\second}, corresponding to a stable opening, was used in the analysis. This observation is consistent with observations from previous MD simulation studies \cite{Li2018}. The conductive selectivity filter of all three states was fully loaded with K$^+$ at the start of the simulation and was found to rapidly evolve to the WKK0KW occupancy, where the symbols W, K and 0 stand for occupacy of the site by a water molecule, a K$^+$ ion, or nothing, respectively (Fig. \ref{SI_SF_occupation}). In the FO state simulation, a water molecule entered the selectivity at t=\SI{400}{\nano\second}. Water penetration in the selectivity filter has been linked to inactivation, particularly in this inactivation-prone wide-gate state\cite{Furini2020}. Therefore the rest of the simulation data was discarded. In contrast, consistent with previous observations\cite{Li2018}, for the C state, no instabilities in the structure or the selectivity filter occupancy were noticed such that the full trajectory length was considered for the analysis. The simulation data for the different states that were used for CS analysis is summarized in figure \ref{SI_SF_occupation}.

\subsection{Chemical Shift Prediction from the MD Results}

We used SPARTA+\cite{Shen2010} and SHIFTX2,\cite{Han2011} two well established tools, to predict the chemical shifts expected for the ensemble of structures observed in the MD trajectories. Predictions were made based on numerous frames from the MD trajectory, and subsequently the average and the distribution in predicted shifts was analyzed, following a procedure analogous to Robustelli et al.\cite{Robustelli2012} The two tools (SPARTA+ and SHIFTX2) produced chemical shifts in qualitative agreement; however, the predicted chemical shifts were not identical for the two tools (Figure \ref{SI_SPARTA_v_SHIFTX2}). The results from both methods were thus analyzed separately to ensure robustness of the conclusion. We note that direct comparison of the CS predicted directly from the XRD structures does not give a clear picture of which state is closer to experiment (Fig. \ref{SI_assignment_XRD}).

The distribution of the predicted CS based on the MD simulation trajectory is strikingly non-Gaussian as can be seen in Figures \ref{SI_hist1} - \ref{SI_hist8}. This fact  introduces difficulty in  assessing the statistical significance of differences, and a need for a robust method to assess the statistical significance of changes in CS between simulated states. Therefore, we chose to model the CS distributions of the simulation using Bayesian inference (Fig. \ref{general_scheme}b). 

The goal of Bayesian inference is to numerically calculate the \textit{posterior} distribution of the parameters of the distribution of CS shifts (chosen here as a skew-normal distribution, see Methods, Figure \ref{general_scheme}b), p($\mu,\sigma,\alpha \vert $d), given our data (which is here the CS predictions from MD snapshots). The posterior distribution is thus a measure of the uncertainty of the CS shifts distribution parameters considering the data we have collected. Note that we assign uncertainty to our knowledge of the parameters and not to the data directly, contrary to a classical statistics approach in which parameters that we assume to be certain are estimated, and then the probability to obtain the data sampled calculated by a p-value given the obtain parameters. 

To calculate the posterior probability we must choose the terms of the right hand side of Bayes law. The \textit{likelihood} term or probability of the data given the parameters, p($d \vert \mu,\sigma,\alpha$), is specified by choosing the distribution with which we model our data presented in the central column of Figure \ref{general_scheme}b. After the inference has been carried out, the posterior distribution of the parameters can be sampled to obtain the ensemble of skew-normal distributions inferred. This ensemble can be compared to the data to see the adequateness of the model. This procedure is known as \textit{posterior predictive check} (Figure \ref{SI_ppc}). As Figures \ref{SI_hist1} - \ref{SI_hist8} shows, our data is adequately characterized by skew-normal distributions. A skew-normal distribution can be defined by three parameters: the location ($\mu$), the scale parameter ($\sigma$), and skewness parameters ($\alpha$).

Finally the posterior distributions are estimated numerically using the NUTS\cite{sanjeevi2017nuts} algorithm. This algorithm also calculates the normalization factor $p(d)$ or the probability of the data. Once these posterior distributions are known, we can calculate the probability distributions of the mean of the skew-normal distribution $\big(\mu+\sigma\sqrt{\frac{2}{\pi}}\frac{\alpha}{\sqrt{1+\alpha^2}}\big)$, which we ascribe to the NMR-peak position and call CS$_{\text{sim}}^{\text{X}}$, where X is the state simulated. CS$_{\text{sim}}^{\text{X}}$, like all quantities obtained with Bayesian inference, is thus not assigned a single number but rather a probability distribution. As a consequence, for simplicity, the distributions of these random variables can be summarized by the interval that contains 94$\%$ of its probability. This interval is known as the credible interval (CI) and is written as its center plus or minus the distance to the bounds of the interval. This is analogous to the confidence intervals in classical statistics.

\subsection{Activated State Characterization Through Statistical Analysis}

We approached the question of what structure best represents the activated samples studied by NMR spectroscopy by asking two questions. First, we asked which chemical shifts are predicted to distinguish the FO and the PO states. Then we asked whether the NMR shifts corresponding to the activated state suggest better agreement with the FO or the PO configuration.

Many of the predicted ${\text{CS}}_{\text{sim}}^{\text{X}}$ shifts do not change sufficiently between the PO and FO states to offer any statistical power of discrimination (Figure \ref{SI_corr1}, \ref{SI_corr2}). We thus developed the following protocol to identify nuclei capable of discriminating between the two states.

The difference between the simulated chemical shifts of both states ($\text{CS}_{\text{sim}}^{\text{FO}}-\text{CS}_{\text{sim}}^{\text{PO}}$) and the effect size of this difference $\big(\frac{\text{CS}_{\text{sim}}^{\text{FO}}-\text{CS}_{\text{sim}}^{\text{FO}}}{\text{std}_{pool}}\big)$, where $\text{std}_{pool} =\sqrt{\frac{var(\text{CS}_{\text{sim}}^{\text{FO}})-var(\text{CS}_{\text{sim}}^{\text{PO}})}{2}}$ and $var$ is the variance function, were calculated. The first expression estimated whether there was a difference between the CS predicted for both states and the second if this difference was large compared to the variance of the CS distributions. A graphical interpretation of this procedure can be found in Figure \ref{general_scheme}c. Due to the effect size condition for statistical significance, this methodology relies on having a chemical shift distribution per nucleus and can only be performed having an ensemble of protein configurations obtained from statistical simulation. In addition, this filtering relies solely on simulated data.

An example of  effect size and difference in means for \ca and the chemical shift prediction method SPARTA+ is presented in Figure \ref{Fig3}. In this figure, we present the difference in means (left graph) and the effect size (right graph) for  amino acids whose experimental \ca CS have been assigned in both activated and deactivated states. 

A CS variable of a given nucleus in a given residue and generated by a given chemical shift prediction method is considered as having discrimination power if the center of the  CS difference CI is greater in absolute value than the experimental tolerance (0.2 ppm for C, \ca, \cb and 0.5 ppm for N) and the center of the effect size CI is larger than 0.5 in absolute value. This procedure is repeated independently for each studied nucleus, for each chemical shift prediction method, and for each residue (Figures \ref{SI_stat_filt_1}, \ref{SI_stat_filt_2}). 14 \ca nuclei, 11 \cb, 11 C, and 13 N nuclei were found to fit these criteria, yielding a total of 49 nuclei with potential to distinguish the two open states.

\begin{figure*}[tbp]
\includegraphics[width=0.7\textwidth]{statistical_filtering_skew_model_sparta_plus_main.jpeg}% Here is how to import EPS art
\caption{\label{Fig3} \scriptsize
    Statistical filtering of simulated CS.  Circles represent the center of the 94$\%$ credible interval of the variable distribution and horizontal bars represent the full range of the corresponding credible interval (which are not visible in cases where the interval is smaller than or comparable to the symbol size). The vertical axis represents the residues that are part of the experimental dataset in both pH conditions. We show here CS differences for the \ca nuclei using the SPARTA+ chemical shift prediction method. Left: credible interval of $\text{CS}_{\text{sim}}^{\text{X}}$ difference in means between the Fully Open (FO) and Partially Open (PO) states. If a credible interval center is outside the experimental tolerance range (green shade), the difference is considered significant. Right: credible interval of the effect size (difference in means scaled by the pooled standard deviation) between the FO and PO states. If the credible interval center is greater than 0.5 in absolute value (green shade), the effect size is considered medium-large\cite{Sawilowsky2009}. A residue must have a significant difference in means and a large effect size to be considered as having discrimination power. The same procedure was carried out for the rest of the nuclei and chemical shift prediction methods (Figs. \ref{SI_stat_filt_1},\ref{SI_stat_filt_2})}
\end{figure*}

As mentioned above, the strategy developed in this work involved analysis of open-vs-closed or activated-vs-deactivated difference shifts rather than absolute chemical shifts.  Specifically, the  experimental chemical shift changes between the activated  and deactivated states ($\Delta\text{CS}_{\text{exp}}$) was compared to the predicted chemical shifts changes between the FO/PO vs the C simulation MD  ensembles ($\Delta\text{CS}_{\text{sim}}^{\text{X}}$).   An overview of the method can be found in Figure \ref{general_scheme}d. Such a strategy of calculating difference chemical shifts is likely to be less prone to a number of the errors  that make calculations of absolute shifts very challenging. For example, CS referencing errors will not be present in the difference shift calculation.  Also, consistent effects that occur in the experimental data but are not captured by the prediction algorithm (e.g. effects of the lipid environment, or tertiary interactions) will presumably cancel and produce less systematic error in the calculation of difference shifts if the states (open vs closed) are relatively similar. Indeed, the use of $\Delta\Delta\text{CS}^{\text{X}}$ ($\Delta\text{CS}_{\text{sim}}^{\text{X}}$ - $\Delta\text{CS}_{\text{exp}}$) has allowed us to make stronger predictions than absolute CS (CS$_{\text{sim}}$ - $\text{CS}_{\text{exp}}$) would have (Figure \ref{SI_assignment_no_ref}). Additionally, the error distribution of the $\Delta\Delta\text{CS}^{\text{X}}$ is narrower and more centered at zero than the analogous distribution of absolute chemical shifts (Figure \ref{SI_error_dist}) implying that the use of $\Delta\Delta\text{CS}^{\text{X}}$ has smaller random and systematic errors and is thus is more suited for state prediction than absolute chemical shifts. Alternatively, correcting the center of these distributions (CS$_{\text{sim}}$ - $\text{CS}_{\text{exp}}$) by fitting to a Gaussian distribution (Figures \ref{SI_sparta_histogram}-\ref{SI_shiftx2_histogram}) and then shifting the center of this Gaussian distribution to zero yields a small improvement over the absolute CS difference (Figure \ref{SI_assignment_shift}). This strategy of computing difference shifts may be suitable only to specific situations. Considering absolute chemical shift predictions appears to be preferable if the system undergoes major conformational transitions or if only one state of the system is accessible experimentally. The approach we propose should instead be considered when the structure of a reference state is available and changes in the system are relatively subtle, with many structural features preserved between the various conformational states of the system. 

Figure \ref{state_assignment_scheme}a shows the $\Delta\Delta\text{CS}^{\text{X}}$ for the PO (blue bars) and FO states (orange bars) for the different nuclei and predicted using the two different prediction software tools. Missing bars correspond to an absence of experimental CS signal or to a statistical non-significance of the simulated value difference between states (Figure \ref{SI_assignment_all}). The lower the $\Delta\Delta\text{CS}^{\text{X}}$, the closer the simulation prediction is to the experiment. Therefore if, for a given nucleus, a given method, and a given residue, the bar corresponding to the PO state is lower than the bar corresponding to the FO state, we consider the CS$^\text{PO}_{\text{sim}}$ of the PO state to agree better with the CS$^\text{pH 4, act}_{\text{exp}}$ of the activated state. If both signals are below the green shaded area, which indicates the experimental tolerance, a most likely state is not assigned. Most of the 49 distinguishing nuclei identified in this study indicate a better agreement with the PO state (61$\%$) than with the FO state (25$\%$), suggesting that the activated samples prepared for NMR analysis more closely agree with a partial opening of the activation gate. If instead we consider which state the majority of the nuclei of a residue indicate, these percentages change to 59\%, 9\% and 32\% for PO, FO, and undetermined respectively.

\begin{figure}[tbp]
\includegraphics[width=\textwidth, trim={ 0 10cm  0 0},clip]{state_assignment_scheme.pdf}% Here is how to import EPS art
\caption{\label{state_assignment_scheme} \scriptsize
    (a) Centers of 94$\%$ credible intervals of the difference in relative chemical shifts between experiment and simulation in absolute value. The limits of the credible interval are shown as error bars. Both experiment and simulation use as reference the closed state chemical shifts: $\Delta\text{CS}_{\text{sim}}^{\text{X}}$ = CS$_{\text{sim}}^{\text{X}}$ - CS$_{\text{sim}}^{\text{C}}$  and $\Delta\text{CS}_{\text{exp}}$ = CS$_{\text{exp}}^{\text{pH }4}$ - CS$_{\text{exp}}^{\text{pH }7.5}$, where X represents the FO or PO sate. The residues found to be capable of discriminating between PO and FO using our statistical criteria are represented on the x-axis for the different nuclei and chemical shift prediction methods. The agreement between MD simulations and the NMR experiments is higher the closer the value is to zero. The PO state (blue) has in general a better agreement with experiment than FO state (orange). The green shade depicts the typical experimental uncertainty. If two methods produce statistically identical CS or the signal is missing in the spectrum, the bar is not represented. (b) Side view of the molecular structure of the PO state. Only two subunits are shown. Residues identified using one chemical shift prediction method are shown with a thin "licorice representation". Residues identified using two chemical shift prediction methods are shown with a thick "licorice representation". (c) Molecular structure of the PO state viewed from the intracellular side. (d) and (e) show the same representations as (b) and (c) but for the FO state.
    (f) KcsA sequence, truncated to region of the protein used in the MD simulations. Blue highlights represent residues whose experimental chemical shift agrees with the simulated PO state, orange highlights those in agreement with the FO state and grey highlights inconclusive residues.} 
\end{figure}

The nuclei displaying better agreement with the PO state (highlighted in blue in Figure \ref{state_assignment_scheme}b,c) are located in the turret region linking TM1 and the pore helix (A50, E51, G56, Q58, L59, I60), in the selectivity filter (T75, Y78), and in the transmembrane part of TM2 (M96, V97, S102, F103, T106 and T112). Additional nuclei spread all over the sequence were inconclusive (grey highlights in Figure \ref{state_assignment_scheme}c - Y62, R64, R88, V93, A98, A108 and A109). Only two residues displayed better agreement with the FO state (orange highlights in Figure \ref{state_assignment_scheme}b,c - T61 and G99).

Within the approach of analyzing the difference in predicted and experimental chemical shifts, one can imagine a number of logical ways to compare the various structural states or MD ensembles (FO, PO, and C) to the two experimental states (activated and deactivated). We developed an \textit{ad hoc} classification system summarized briefly in Figure \ref{FlowChart} and in more detail in Figure \ref{SI_FlowChart}. Individual nuclei are first grouped in 5 classes based on the predicted shifts for the three MD ensembles, as follows: (\textbf{A}) a very well populated group for which the three structural states are predicted to give rise to indistinguishable shifts (akin to experimental spectators), (\textbf{B}) a well populated group for which the FO state is predicted to be distinct from the C and the PO states (which are themselves not distinguishable from each other); (\textbf{C}) a poorly populated group for which, curiously, PO is distinct from FO and C (which are themselves not distinguishable from each other); (\textbf{D}) an unpopulated hypothetical group for which the C state is distinct but PO and FO are indistinguishable ($\Delta\text{CS}_{\text{sim}}^{\text{FO}}$ $\sim$ $\Delta\text{CS}_{\text{sim}}^{\text{PO}}$), which has been eliminated by our statistical filtering analysis; and (\textbf{E}) a fairly well populated group for which all three computed states are expected to give rise to distinct chemical shifts. Each nucleus is further classified in terms of whether (\textbf{1}) the two experimental states have indistinguishable chemical shifts, or (\textbf{2}) if a change in the chemical shift is observed. Each nucleus is then identified as being category \textbf{B1}, \textbf{B2},… up to \textbf{E2} (and in some cases is identified as belonging to two classes because the two chemical shift prediction tools that were used were not in agreement).

\begin{figure}[tbp]
\includegraphics[width=0.45\textwidth]{Markers_FlowChart_v4-02.eps}
\caption{\label{FlowChart} \scriptsize
    Flow chart demonstrating the combinations of the $\Delta\text{CS}_{\text{sim}}^{\text{X}}$ and $\Delta\text{CS}_{\text{exp}}$ that yield various classifications of the state markers identified in this study. A threshold of tolerance of 0.2 ppm for \ce{^13C} and 0.5 ppm for \ce{^15N} was used to determine if the CS change was significant. FO = fully open, PO = partially open, C = closed, $\Delta\text{CS}_{\text{sim}}^{\text{X}}$ = $\text{CS}_{\text{sim}}^{\text{X}}$ - $\text{CS}_{\text{sim}}^{\text{Closed}}$, and $\Delta\text{CS}_{\text{exp}}$ = $\text{CS}_{\text{exp}}^{\text{pH } 4\text{, act}}$ - $\text{CS}_{\text{exp}}^{\text{pH } 7.5\text{, deact}}$.}
\end{figure}

Two classes of nuclei are of particular note in this study. A sparsely populated type, \textbf{E2}, where the predicted CS is different for each state and the experimental data for the nucleus also indicates a shift, has strong ability to address the questions posed in this work. (This class includes A50C, E51C, T107\ca, T107N, A109C, and A109\cb). These markers clearly show a  better agreement with the conclusion that the sample is PO rather than FO (Figure \ref{state_assignment_scheme}a). Nuclei classified as \textbf{B1}, for which the FO state (but not the PO state) is predicted to show a deviation from C, and for which the experimental data obtained at both pH conditions are indistinguishable, are the most common (apart from the spectators \textbf{A}, which are not discussed here). Resonances that fall into the \textbf{B1} classification in aggregate offer strong support for the conclusion that the activated samples studied experimentally are closer to PO than to FO (A50\ca, G56\ca, G56C, Q58\ca, L59N, I60\cb, R64\cb (SPARTA+), T75\ca, Y78\ca, Y78\cb, R89\ca, M96\ca, M96\cb, M96C, M96N, V97\cb, V97C, V97N, G99N, S102\ca, and A108N).
 
Another group of CS are classified as \textbf{B2}, when the FO state is predicted to differ from the PO and C states, and the experimental data do show a shift (Q58C, A97\cb, G99C, S102N, and A108C) and yet another as \textbf{E1} for which the predicted CS differs for each structural state while the data do not reflect a change in CS (L59C, I60N, T61\ca (SPARTA+), Y62\cb, V93\ca, A98\ca (SHIFTX2), and F103N). These two types of markers could be inconclusive although in some cases there is a much better match for one hypothesis than the other (typically the PO state being in better agreement). 
 
A handful of nuclei are observed as class \textbf{C1} if the predicted PO shifts are different from the FO and C states while the experimental data show no shift. At first sight, these would appear to indicate that the sample is FO. The pattern is difficult to interpret mechanistically and the phenomenon is mainly observed in the turret region for which the connection to opening is unclear and where antibody binding in the crystal structure may have complicated the MD sampling (T61\ca (SHIFTX2), T61\cb, T61N, Y62N, R64\ca (SHIFTX2), R64\cb, V93N, A98\ca (SPARTA+), and G99N).

No resonance is classified into \textbf{C2}, where the PO shift is distinct from the other states and there is an observed experimental chemical shift change between the activated and deactivated states. This particular class would suggest that the resonances that are identified within it would favor the sample to be in the PO state. Notably, some additional classifications in this study are populated by no resonances, namely \textbf{D1} and \textbf{D2}, due to the initial conditions of the analysis that require a resonance be distinguishable between the FO and PO states.


\subsection{Molecular Basis for KcsA activation}

The chemical shift markers identified by the SSNMR experiments reflect conformational gating of the corresponding residue during opening. These sites exhibit a variety of conformational differences in the MD simulations, including sidechain rotamer hops and backbone rearrangements (especially at the TM2 hinge) (Figure \ref{bio}).

Several of the residues that enabled the distinction between the PO and FO states (A50, A98, S102, T107, A108 and A109) also exhibit experimental chemical shift changes when comparing deactivated and activated datasets (Figure \ref{FlowChart}). S102 and T107, which undergo rotameric changes in their $\chi_1$ dihedral angles, in particular, offer particularly strong support to the conclusion of partial opening (Figure \ref{bio}d,h). 

\begin{figure}[tbp]
\includegraphics[width=0.75\textwidth]{bio.pdf}% Here is how to import EPS art
\caption{\label{bio} 
\scriptsize
    Structural comparison between C, PO, and FO states (a) Side view of 2 opposite subunits in the C (green ribbon), PO (blue) and FO states (orange). The selectivity filter region is highlighted by a grey box, the hinge region by a green box and the cavity facing helices by a purple box. (b) Dihedral (Ramachandran and $\chi_1$-$\chi_2$ plots) angle plots showing the MD simulation distributions (surface contours, coloring is the same as in panels (a) and (c) corresponding molecular models showing details of the environment around Y78 in the three states. Snapshots were aligned on the selectivity filter residues backbone (75-80) of the subunit Y78 belongs to. (d-e) Same as (c) for S102 (f-i) same as (c) for F103 and T107. The molecular views in the purple boxes (g,i) show the rearrangement of the rotameric state of T107 from the crystal structure to the most prevalent rotameric state observed in MD simulations, in the FO state.} 
\end{figure}

Scrutinizing additional dihedral angle plots revealed other substantial rearrangements at residues identified to discriminate the PO and the FO states in the selectivity filter (Y78), in the hinge region (M96, V97, A98, and S102) and in the residues facing the central pore cavity (F103 and T107) (Figure \ref{bio}). The hinge region residues displayed subtle changes in the alpha helical basin, as expected from the hinge motion that is key for gate opening described in numerous studies (Figure \ref{bio}d-i).\cite{Cuello2017,Li2018} F103 has indeed been reported as a key player in transmitting the allosteric signal coupling the activation gate and the inactivation gate at the selectivity filter.\cite{Li2018} In the C and PO states, several rotameric states are accessible to the $\chi_1$ angle of this residue, while in the FO state the residue is locked in the state found in the XRD structure avoiding exposure to the hydrated cavity. Several studies have found that its clash with T74 leads to SF pinching linked to inactivation \cite{Li2018,kopec2019molecular}. Our work seems to indicate that in high [\ce{K+}] and low pH conditions, locking of this residue in the rotameric state compatible with inactivation is not favored. T107 assumes a similar rotameric state in the C, PO and FO state XRD structures (Figure \ref{bio}h). Intriguingly, our MD simulations show that another $\chi_1$ dihedral conformation is more prevalent in the FO state. This FO rotameric state appears unfavorable in the activated state condition SSNMR. While the molecular basis for this observation remains unclear, we note that a CS prediction based on a pure XRD structure diffraction (in the absence of MD simulations) would have missed this observation.

The only SF residue with discriminating power which shows an interesting rotameric change is Y78 (Figure \ref{bio}b). Indeed the other SF residue, T75, is a weak marker, with the \ca nucleus CS pointing towards the PO state only when predicted by SPARTA+. Y78 is at the center of a hydrogen bonding hub which connects the selectivity filter and the pore helix of adjacent subunits, via hydrogen bonding to the side chains of T72 and W68 and the backbone carbonyl of E71 (Figure \ref{bio}c). The rotameric state of Y78 $\chi_2$ differs between the XRD structures of the C and PO state, on one hand, and of the FO state, on the other (Figure \ref{bio}b). The slight change in Y78 orientation in the FO state appears to lead to a motion of the pore helix. Interestingly no pore helix residues, including T72 and W68, showed a change in predicted CS between the PO and the FO states.

Only two residues indicated a better agreement with the FO state, T61 and G99. It would be intriguing if a portion of the channel were in a partially open like state and another portion were fully open, but this seems not likely to be quite correct. For example T61 is a more complex situation without a clear assignment to either state. A comparison of the Ramachandran distributions of T61 in both states reveals that both states explore a common conformer but the PO state samples an additional conformer (Fig. \ref{SI_dihedral}). This conformer, which is only present in the PO state simulations, is only explored by one of four subunits. If the chemical shifts produced from this subunit are removed then the difference in CS between PO and FO states becomes statistically insignificant (Fig. \ref{SI_stat_filt_T61}). We attribute this conformational state to a fluctuation in the dynamics of the flexible turret region which may not be representative of the entire configurational ensemble. This region was indeed in contact with the antibodies used to stabilized the sample in the XRD structure and numerous indications in the literature suggest dynamic plasticity for these residues.\cite{Cuello2010}  G99 is located in the center of the hinge region, surrounded by residues that exhibit good agreement with the PO state.  The fact that its chemical shift agrees better with  the FO state appears intriguing. Indeed, the dihedral angles explored by G99 in both states are fairly similar and correspond to the initial angles of the X-ray structure (Fig. \ref{SI_dihedral}). Why it is the only residue in better agreement with the FO state in this region remains to be explained.



\section{Conclusions}

The open conductive state of KcsA has remained elusive to structural studies due to its short lifetime. Mutagenesis and other engineering strategies have led to the determination of several XRD structures, but since they were obtained under somewhat artificial conditions, the physiologically relevant state remains of interest to be fully characterized. Spectroscopy studies have revealed that there is considerable conformational heterogeneity within the open states of KcsA of the degree of opening, making a static characterization not only challenging but also limited in its physiological relevance. SSNMR allows the study of a sample at near-physiological conditions, but interpreting the spectra in terms of structure is challenging. MD simulations can assist with this interpretation since the conditions of the MD simulation and the SSNMR experiment can be nearly equivalent, and the time-scales probed are compatible. This procedure rests on the use of chemical shift prediction software which have been refined over the years to yield robust predictions.

Here, we have designed a computational pipeline to characterize the activated state present in the SSNMR activated state sample. The two candidates from X-ray crystallography that we compared it to were a partially open PO (PDB ID 3FB5) and a fully open FO (PDB ID 5VK6) structure. While a direct comparison between the experimental and computationally predicted CS was noisy, implementing a Bayesian inference scheme in which nuclei were selected when they had the predictive power to statistically discriminate the PO and the FO states, and the use of difference chemical shifts (relative to the deactivated/closed state) for both the experimental and the predicted shifts, proved more successful. With this approach, we were able to conclude that the majority state in the SSNMR resembles the PO state. 

Why does the system stall in the partially open state, instead of advancing to form the inactivated state as it does in electrophysiology experiments? Formation of the fully open state may linked to ion expulsion, and so may be prevented in these SSNMR experiments by the relatively high concentration of ambient potassium. In other words, perhaps the FO state is unstable in the presence of bound potassium ions.  Indeed, under some conditions the FO state characterized by X-ray crystallography exhibited reduction in the ion occupancy in the selectivity filter.\cite{Cuello2010}  MD simulations of the opening also appear to indicate a slow process from partial opening to full opening,  with an unknown inherent bottleneck, of a possibly entropic nature, or it is possible that the system does not fully open absent other coupled processes such as ion loss. Alternatively, perhaps the various differences between the sample conditions (protein concentration, symmetric vs. asymmetric lipids and pH, etc.) cause the system to stall in one case and advance to inactivation in the other. 

The conclusion that the activated state studied by SSNMR is a partially open state seems compatible with a number of earlier studies or hypotheses. It has been speculated in prior studies that the state stabilized for NMR studies by cardiolipin lipids at low pH is partially open.\cite{VanderCruijsen2017} It is also consistent with the fact that preparation of the FO state for  X-ray crystallography experiments required a number of mutations and cross links, without which the partially open state resulted. The results collectively suggest a mechanism in which opening, once triggered by protonation on the pH gate, proceeds to an intermediate point which can be stabilized under appropriate experimental conditions such as high \ce{K+} concentration.  The degree of opening is possibly a balancing act between repulsions in the protonated pH sensor that tend to push the system open, and conformational clashes in the hinge and base of the selectivity filter, which tend to keep the system closed. 

This work demonstrates an approach to combining  SSNMR and MD simulations to provide a molecular level description of a physiologically important allosteric process in a membrane protein.

  
\section{Supplementary Materials}
See Supplementary Materials for additional data figures and tables. 

\section{Dedication}
We dedicate this paper to Professor Cynthia Jameson of UIC for her contributions to fundamental understanding of the NMR chemical shift, mapping the NMR Chemical Shift and coupling constants across the periodic table, and providing insights into intermolecular effects, and the importance of considering ensembles in calculating shifts.  This dedication recognizes Cynthia’s many contributions to furthering equity and diversity in science, especially for women scientists.

\section{Acknowledgements}
E.G.K. was supported by a postdoctoral fellowship from the National Institutes of Health (NIH) (Grant No. GM135350). A.E.M. was supported by Grant No. GM088724 from the NIH and Grant No. 1412253 from the National Science Foundation. The NMR data were collected at the New York Structural Biology Center (NYSBC) with support from the Center on Macromolecular Dynamics by NMR Spectroscopy, a Biomedical Technology Research Resource (NIH Grant No. GM118302). The NYSBC is also supported by the Empire State Division of Science Technology and Innovation and the Office of Research Infrastructure Program (NIH Grant No. CO6RR015495). A.E.M. is a member of the NYSBC.  S.P.C. was supported by a postdoctoral scholarship of the Gustafsson foundation. L. D. would like to thank the support of Science for Life Laboratory, the Göran Gustafsson Foundation, and the Swedish research council (Grant No. VR-2018-04905). Simulations were performed on computational resources provided by the Swedish National Infrastructure for Computing (SNIC) at the PDC Centre for High Performance Computing (PDC-HPC). The authors thank Zhiyu Sun for the synthesis of one of the KcsA samples used in this work. The authors thank Dr. Keith Fritzsching for helpful discussions.

\section{Data Availability}

The data that support the findings of this study are openly available in \url{https://github.com/delemottelab/Informing_NMR_experiments_w_MD} and \url{https://osf.io/6h2z5/} with the raw NMR data deposited in BMRbig (\url{https://bmrbig.org}) with under deposition ID bmrbig9.

%\nocite{*}
% \bibliography{bibliography}% Produces the bibliography via BibTeX.
%merlin.mbs aipnum4-1.bst 2010-07-25 4.21a (PWD, AO, DPC) hacked
%Control: key (0)
%Control: author (8) initials jnrlst
%Control: editor formatted (1) identically to author
%Control: production of article title (0) allowed
%Control: page (1) range
%Control: year (1) truncated
%Control: production of eprint (0) enabled
\providecommand{\noopsort}[1]{}\providecommand{\singleletter}[1]{#1}%
\begin{thebibliography}{71}%
\makeatletter
\providecommand \@ifxundefined [1]{%
 \@ifx{#1\undefined}
}%
\providecommand \@ifnum [1]{%
 \ifnum #1\expandafter \@firstoftwo
 \else \expandafter \@secondoftwo
 \fi
}%
\providecommand \@ifx [1]{%
 \ifx #1\expandafter \@firstoftwo
 \else \expandafter \@secondoftwo
 \fi
}%
\providecommand \natexlab [1]{#1}%
\providecommand \enquote  [1]{``#1''}%
\providecommand \bibnamefont  [1]{#1}%
\providecommand \bibfnamefont [1]{#1}%
\providecommand \citenamefont [1]{#1}%
\providecommand \href@noop [0]{\@secondoftwo}%
\providecommand \href [0]{\begingroup \@sanitize@url \@href}%
\providecommand \@href[1]{\@@startlink{#1}\@@href}%
\providecommand \@@href[1]{\endgroup#1\@@endlink}%
\providecommand \@sanitize@url [0]{\catcode `\\12\catcode `\$12\catcode
  `\&12\catcode `\#12\catcode `\^12\catcode `\_12\catcode `\%12\relax}%
\providecommand \@@startlink[1]{}%
\providecommand \@@endlink[0]{}%
\providecommand \url  [0]{\begingroup\@sanitize@url \@url }%
\providecommand \@url [1]{\endgroup\@href {#1}{\urlprefix }}%
\providecommand \urlprefix  [0]{URL }%
\providecommand \Eprint [0]{\href }%
\providecommand \doibase [0]{http://dx.doi.org/}%
\providecommand \selectlanguage [0]{\@gobble}%
\providecommand \bibinfo  [0]{\@secondoftwo}%
\providecommand \bibfield  [0]{\@secondoftwo}%
\providecommand \translation [1]{[#1]}%
\providecommand \BibitemOpen [0]{}%
\providecommand \bibitemStop [0]{}%
\providecommand \bibitemNoStop [0]{.\EOS\space}%
\providecommand \EOS [0]{\spacefactor3000\relax}%
\providecommand \BibitemShut  [1]{\csname bibitem#1\endcsname}%
\let\auto@bib@innerbib\@empty
%</preamble>
\bibitem [{\citenamefont {Cournia}\ and\ \citenamefont
  {Chatzigoulas}(2020)}]{Cournia2020}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {Z.}~\bibnamefont
  {Cournia}}\ and\ \bibinfo {author} {\bibfnamefont {A.}~\bibnamefont
  {Chatzigoulas}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{Allostery
  in membrane proteins}},}\ }\href {\doibase 10.1016/j.sbi.2020.03.006}
  {\bibfield  {journal} {\bibinfo  {journal} {Current Opinion in Structural
  Biology}\ }\textbf {\bibinfo {volume} {62}},\ \bibinfo {pages} {197--204}
  (\bibinfo {year} {2020})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Xu}, \citenamefont {Bhate},\ and\ \citenamefont
  {McDermott}(2017)}]{Xu2017}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {Y.}~\bibnamefont
  {Xu}}, \bibinfo {author} {\bibfnamefont {M.~P.}\ \bibnamefont {Bhate}}, \
  and\ \bibinfo {author} {\bibfnamefont {A.~E.}\ \bibnamefont {McDermott}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Transmembrane allosteric
  energetics characterization for strong coupling between proton and potassium
  ion binding in the KcsA channel}},}\ }\href {\doibase
  10.1073/pnas.1701330114} {\bibfield  {journal} {\bibinfo  {journal}
  {Proceedings of the National Academy of Sciences of the United States of
  America}\ }\textbf {\bibinfo {volume} {114}},\ \bibinfo {pages} {8788--8793}
  (\bibinfo {year} {2017})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Li}\ \emph {et~al.}(2018)\citenamefont {Li},
  \citenamefont {Ostmeyer}, \citenamefont {Cuello}, \citenamefont {Perozo},\
  and\ \citenamefont {Roux}}]{Li2018}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {J.}~\bibnamefont
  {Li}}, \bibinfo {author} {\bibfnamefont {J.}~\bibnamefont {Ostmeyer}},
  \bibinfo {author} {\bibfnamefont {L.~G.}\ \bibnamefont {Cuello}}, \bibinfo
  {author} {\bibfnamefont {E.}~\bibnamefont {Perozo}}, \ and\ \bibinfo {author}
  {\bibfnamefont {B.}~\bibnamefont {Roux}},\ }\bibfield  {title} {\enquote
  {\bibinfo {title} {{Rapid constriction of the selectivity filter underlies
  C-type inactivation in the KcsA potassium channel}},}\ }\href {\doibase
  10.1085/jgp.201812082} {\bibfield  {journal} {\bibinfo  {journal} {J. Gen.
  Physiol.}\ }\textbf {\bibinfo {volume} {215}},\ \bibinfo {pages} {1408--1420}
  (\bibinfo {year} {2018})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Cuello}, \citenamefont {Cortes},\ and\ \citenamefont
  {Perozo}(2017)}]{Cuello2017}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {L.~G.}\ \bibnamefont
  {Cuello}}, \bibinfo {author} {\bibfnamefont {D.~M.}\ \bibnamefont {Cortes}},
  \ and\ \bibinfo {author} {\bibfnamefont {E.}~\bibnamefont {Perozo}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{The gating cycle of a K+
  channel at atomic resolution}},}\ }\href {\doibase 10.7554/eLife.28032}
  {\bibfield  {journal} {\bibinfo  {journal} {Elife}\ }\textbf {\bibinfo
  {volume} {6}},\ \bibinfo {pages} {1--17} (\bibinfo {year}
  {2017})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Xu}\ \emph {et~al.}(2019)\citenamefont {Xu},
  \citenamefont {Zhang}, \citenamefont {Rogawski}, \citenamefont {Nimigean},\
  and\ \citenamefont {McDermott}}]{Xu2019}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {Y.}~\bibnamefont
  {Xu}}, \bibinfo {author} {\bibfnamefont {D.}~\bibnamefont {Zhang}}, \bibinfo
  {author} {\bibfnamefont {R.}~\bibnamefont {Rogawski}}, \bibinfo {author}
  {\bibfnamefont {C.~M.}\ \bibnamefont {Nimigean}}, \ and\ \bibinfo {author}
  {\bibfnamefont {A.~E.}\ \bibnamefont {McDermott}},\ }\bibfield  {title}
  {\enquote {\bibinfo {title} {{Identifying coupled clusters of allostery
  participants through chemical shift perturbations}},}\ }\href {\doibase
  10.1073/pnas.1811168116} {\bibfield  {journal} {\bibinfo  {journal} {Proc.
  Natl. Acad. Sci. U. S. A.}\ }\textbf {\bibinfo {volume} {116}},\ \bibinfo
  {pages} {2078--2085} (\bibinfo {year} {2019})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Furini}\ and\ \citenamefont
  {Domene}(2020)}]{Furini2020}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {S.}~\bibnamefont
  {Furini}}\ and\ \bibinfo {author} {\bibfnamefont {C.}~\bibnamefont
  {Domene}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {Critical
  assessment of common force fields for molecular dynamics simulations of
  potassium channels},}\ }\href@noop {} {\bibfield  {journal} {\bibinfo
  {journal} {Journal of Chemical Theory and Computation}\ }\textbf {\bibinfo
  {volume} {16}},\ \bibinfo {pages} {7148--7159} (\bibinfo {year}
  {2020})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Ostmeyer}\ \emph {et~al.}(2013)\citenamefont
  {Ostmeyer}, \citenamefont {Chakrapani}, \citenamefont {Pan}, \citenamefont
  {Perozo},\ and\ \citenamefont {Roux}}]{Ostmeyer2013}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {J.}~\bibnamefont
  {Ostmeyer}}, \bibinfo {author} {\bibfnamefont {S.}~\bibnamefont
  {Chakrapani}}, \bibinfo {author} {\bibfnamefont {A.~C.}\ \bibnamefont {Pan}},
  \bibinfo {author} {\bibfnamefont {E.}~\bibnamefont {Perozo}}, \ and\ \bibinfo
  {author} {\bibfnamefont {B.}~\bibnamefont {Roux}},\ }\bibfield  {title}
  {\enquote {\bibinfo {title} {{Recovery from slow inactivation in K+ channels
  is controlled by water molecules}},}\ }\href {\doibase 10.1038/nature12395}
  {\bibfield  {journal} {\bibinfo  {journal} {Nature}\ }\textbf {\bibinfo
  {volume} {501}},\ \bibinfo {pages} {121--124} (\bibinfo {year}
  {2013})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Sun}\ \emph {et~al.}(2020)\citenamefont {Sun},
  \citenamefont {Xu}, \citenamefont {Zhang},\ and\ \citenamefont
  {McDermott}}]{Sun2020}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {Z.}~\bibnamefont
  {Sun}}, \bibinfo {author} {\bibfnamefont {Y.}~\bibnamefont {Xu}}, \bibinfo
  {author} {\bibfnamefont {D.}~\bibnamefont {Zhang}}, \ and\ \bibinfo {author}
  {\bibfnamefont {A.~E.}\ \bibnamefont {McDermott}},\ }\bibfield  {title}
  {\enquote {\bibinfo {title} {{Probing allosteric coupling in a constitutively
  open mutant of the ion channel KcsA using solid-state NMR}},}\ }\href
  {\doibase 10.1073/pnas.1908828117} {\bibfield  {journal} {\bibinfo  {journal}
  {Proceedings of the National Academy of Sciences of the United States of
  America}\ }\textbf {\bibinfo {volume} {117}},\ \bibinfo {pages} {7171--7175}
  (\bibinfo {year} {2020})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Ader}\ \emph {et~al.}(2009)\citenamefont {Ader},
  \citenamefont {Schneider}, \citenamefont {Hornig}, \citenamefont {Velisetty},
  \citenamefont {Vardanyan}, \citenamefont {Giller}, \citenamefont {Ohmert},
  \citenamefont {Becker}, \citenamefont {Pongs},\ and\ \citenamefont
  {Baldus}}]{Ader2009}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {C.}~\bibnamefont
  {Ader}}, \bibinfo {author} {\bibfnamefont {R.}~\bibnamefont {Schneider}},
  \bibinfo {author} {\bibfnamefont {S.}~\bibnamefont {Hornig}}, \bibinfo
  {author} {\bibfnamefont {P.}~\bibnamefont {Velisetty}}, \bibinfo {author}
  {\bibfnamefont {V.}~\bibnamefont {Vardanyan}}, \bibinfo {author}
  {\bibfnamefont {K.}~\bibnamefont {Giller}}, \bibinfo {author} {\bibfnamefont
  {I.}~\bibnamefont {Ohmert}}, \bibinfo {author} {\bibfnamefont
  {S.}~\bibnamefont {Becker}}, \bibinfo {author} {\bibfnamefont
  {O.}~\bibnamefont {Pongs}}, \ and\ \bibinfo {author} {\bibfnamefont
  {M.}~\bibnamefont {Baldus}},\ }\bibfield  {title} {\enquote {\bibinfo {title}
  {{Coupling of activation and inactivation gate in a K-channel: Potassium and
  ligand sensitivity}},}\ }\href {\doibase 10.1038/emboj.2009.218} {\bibfield
  {journal} {\bibinfo  {journal} {EMBO Journal}\ }\textbf {\bibinfo {volume}
  {28}},\ \bibinfo {pages} {2825--2834} (\bibinfo {year} {2009})}\BibitemShut
  {NoStop}%
\bibitem [{\citenamefont {Labro}\ \emph {et~al.}(2018)\citenamefont {Labro},
  \citenamefont {{Marien Cortes}}, \citenamefont {Tilegenova},\ and\
  \citenamefont {Cuello}}]{Labro2018}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {A.~J.}\ \bibnamefont
  {Labro}}, \bibinfo {author} {\bibfnamefont {D.}~\bibnamefont {{Marien
  Cortes}}}, \bibinfo {author} {\bibfnamefont {C.}~\bibnamefont {Tilegenova}},
  \ and\ \bibinfo {author} {\bibfnamefont {L.~G.}\ \bibnamefont {Cuello}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Inverted allosteric
  coupling between activation and inactivation gates in K+ channels}},}\ }\href
  {\doibase 10.1073/pnas.1800559115} {\bibfield  {journal} {\bibinfo  {journal}
  {Proceedings of the National Academy of Sciences of the United States of
  America}\ }\textbf {\bibinfo {volume} {115}},\ \bibinfo {pages} {5426--5431}
  (\bibinfo {year} {2018})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Delemotte}(2018)}]{Delemotte2018}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {L.}~\bibnamefont
  {Delemotte}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{Opening
  leads to closing: Allosteric crosstalk between the activation and
  inactivation gates in KcsA}},}\ }\href {\doibase 10.1085/jgp.201812161}
  {\bibfield  {journal} {\bibinfo  {journal} {Journal of General Physiology}\
  }\textbf {\bibinfo {volume} {215}},\ \bibinfo {pages} {1356--1359} (\bibinfo
  {year} {2018})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Zhang}\ \emph {et~al.}(2020)\citenamefont {Zhang},
  \citenamefont {Howarth}, \citenamefont {Parkin},\ and\ \citenamefont
  {McDermott}}]{Zhang2020a}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {D.}~\bibnamefont
  {Zhang}}, \bibinfo {author} {\bibfnamefont {G.~S.}\ \bibnamefont {Howarth}},
  \bibinfo {author} {\bibfnamefont {L.~A.}\ \bibnamefont {Parkin}}, \ and\
  \bibinfo {author} {\bibfnamefont {A.~E.}\ \bibnamefont {McDermott}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{NMR studies of lipid
  regulation of the K+ channel KcsA}},}\ }\href {\doibase
  10.1016/j.bbamem.2020.183491} {\bibfield  {journal} {\bibinfo  {journal}
  {Biochim. Biophys. Acta - Biomembr.}\ ,\ \bibinfo {pages} {183491}} (\bibinfo
  {year} {2020})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Zhou}\ \emph {et~al.}(2001)\citenamefont {Zhou},
  \citenamefont {Morais-Cabral}, \citenamefont {Kaufman},\ and\ \citenamefont
  {Mackinnon}}]{Zhou2001}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {Y.}~\bibnamefont
  {Zhou}}, \bibinfo {author} {\bibfnamefont {J.~H.}\ \bibnamefont
  {Morais-Cabral}}, \bibinfo {author} {\bibfnamefont {A.}~\bibnamefont
  {Kaufman}}, \ and\ \bibinfo {author} {\bibfnamefont {R.}~\bibnamefont
  {Mackinnon}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{Chemistry of
  ion coordination and hydration revealed by a K+ channel-Fab complex at 2.0
  {\AA} resolution}},}\ }\href {\doibase 10.1038/35102009} {\bibfield
  {journal} {\bibinfo  {journal} {Nature}\ }\textbf {\bibinfo {volume} {414}},\
  \bibinfo {pages} {43--48} (\bibinfo {year} {2001})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Bhate}\ \emph {et~al.}(2013)\citenamefont {Bhate},
  \citenamefont {Wylie}, \citenamefont {Thompson}, \citenamefont {Tian},
  \citenamefont {Nimigean},\ and\ \citenamefont {McDermott}}]{Bhate2013}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {M.~P.}\ \bibnamefont
  {Bhate}}, \bibinfo {author} {\bibfnamefont {B.~J.}\ \bibnamefont {Wylie}},
  \bibinfo {author} {\bibfnamefont {A.}~\bibnamefont {Thompson}}, \bibinfo
  {author} {\bibfnamefont {L.}~\bibnamefont {Tian}}, \bibinfo {author}
  {\bibfnamefont {C.}~\bibnamefont {Nimigean}}, \ and\ \bibinfo {author}
  {\bibfnamefont {A.~E.}\ \bibnamefont {McDermott}},\ }\bibfield  {title}
  {\enquote {\bibinfo {title} {{Preparation of uniformly isotope labeled KcsA
  for solid state NMR: Expression, purification, reconstitution into liposomes
  and functional assay}},}\ }\href {\doibase 10.1016/j.pep.2013.07.013}
  {\bibfield  {journal} {\bibinfo  {journal} {Protein Expression and
  Purification}\ }\textbf {\bibinfo {volume} {91}},\ \bibinfo {pages}
  {119--124} (\bibinfo {year} {2013})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Wylie}, \citenamefont {Bhate},\ and\ \citenamefont
  {McDermott}(2014)}]{Wylie2014}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {B.~J.}\ \bibnamefont
  {Wylie}}, \bibinfo {author} {\bibfnamefont {M.~P.}\ \bibnamefont {Bhate}}, \
  and\ \bibinfo {author} {\bibfnamefont {A.~E.}\ \bibnamefont {McDermott}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Transmembrane allosteric
  coupling of the gates in a potassium channel}},}\ }\href {\doibase
  10.1073/pnas.1319577110} {\bibfield  {journal} {\bibinfo  {journal}
  {Proceedings of the National Academy of Sciences of the United States of
  America}\ }\textbf {\bibinfo {volume} {111}},\ \bibinfo {pages} {185--190}
  (\bibinfo {year} {2014})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Medeiros-Silva}\ \emph {et~al.}(2016)\citenamefont
  {Medeiros-Silva}, \citenamefont {Mance}, \citenamefont {Dani{\"{e}}ls},
  \citenamefont {Jekhmane}, \citenamefont {Houben}, \citenamefont {Baldus},\
  and\ \citenamefont {Weingarth}}]{Medeiros-Silva2016}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {J.}~\bibnamefont
  {Medeiros-Silva}}, \bibinfo {author} {\bibfnamefont {D.}~\bibnamefont
  {Mance}}, \bibinfo {author} {\bibfnamefont {M.}~\bibnamefont
  {Dani{\"{e}}ls}}, \bibinfo {author} {\bibfnamefont {S.}~\bibnamefont
  {Jekhmane}}, \bibinfo {author} {\bibfnamefont {K.}~\bibnamefont {Houben}},
  \bibinfo {author} {\bibfnamefont {M.}~\bibnamefont {Baldus}}, \ and\ \bibinfo
  {author} {\bibfnamefont {M.}~\bibnamefont {Weingarth}},\ }\bibfield  {title}
  {\enquote {\bibinfo {title} {{\ce{^1H}-Detected Solid-State NMR Studies of
  Water-Inaccessible Proteins In Vitro and In Situ}},}\ }\href {\doibase
  10.1002/anie.201606594} {\bibfield  {journal} {\bibinfo  {journal}
  {Angewandte Chemie - International Edition}\ }\textbf {\bibinfo {volume}
  {55}},\ \bibinfo {pages} {13606--13610} (\bibinfo {year} {2016})}\BibitemShut
  {NoStop}%
\bibitem [{\citenamefont {Jekhmane}\ \emph {et~al.}(2019)\citenamefont
  {Jekhmane}, \citenamefont {Medeiros-Silva}, \citenamefont {Li}, \citenamefont
  {K{\"{u}}mmerer}, \citenamefont {M{\"{u}}ller-Hermes}, \citenamefont
  {Baldus}, \citenamefont {Roux},\ and\ \citenamefont
  {Weingarth}}]{Jekhmane2019}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {S.}~\bibnamefont
  {Jekhmane}}, \bibinfo {author} {\bibfnamefont {J.}~\bibnamefont
  {Medeiros-Silva}}, \bibinfo {author} {\bibfnamefont {J.}~\bibnamefont {Li}},
  \bibinfo {author} {\bibfnamefont {F.}~\bibnamefont {K{\"{u}}mmerer}},
  \bibinfo {author} {\bibfnamefont {C.}~\bibnamefont {M{\"{u}}ller-Hermes}},
  \bibinfo {author} {\bibfnamefont {M.}~\bibnamefont {Baldus}}, \bibinfo
  {author} {\bibfnamefont {B.}~\bibnamefont {Roux}}, \ and\ \bibinfo {author}
  {\bibfnamefont {M.}~\bibnamefont {Weingarth}},\ }\bibfield  {title} {\enquote
  {\bibinfo {title} {{Shifts in the selectivity filter dynamics cause modal
  gating in \ce{} channels}},}\ }\href {\doibase 10.1038/s41467-018-07973-6}
  {\bibfield  {journal} {\bibinfo  {journal} {Nature Communications}\ }\textbf
  {\bibinfo {volume} {10}},\ \bibinfo {pages} {1--12} (\bibinfo {year}
  {2019})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Cuello}\ \emph {et~al.}(2010)\citenamefont {Cuello},
  \citenamefont {Jogini}, \citenamefont {Cortes}, \citenamefont {Pan},
  \citenamefont {Gagnon}, \citenamefont {Dalmas}, \citenamefont
  {Cordero-Morales}, \citenamefont {Chakrapani}, \citenamefont {Roux},\ and\
  \citenamefont {Perozo}}]{Cuello2010}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {L.~G.}\ \bibnamefont
  {Cuello}}, \bibinfo {author} {\bibfnamefont {V.}~\bibnamefont {Jogini}},
  \bibinfo {author} {\bibfnamefont {D.~M.}\ \bibnamefont {Cortes}}, \bibinfo
  {author} {\bibfnamefont {A.~C.}\ \bibnamefont {Pan}}, \bibinfo {author}
  {\bibfnamefont {D.~G.}\ \bibnamefont {Gagnon}}, \bibinfo {author}
  {\bibfnamefont {O.}~\bibnamefont {Dalmas}}, \bibinfo {author} {\bibfnamefont
  {J.~F.}\ \bibnamefont {Cordero-Morales}}, \bibinfo {author} {\bibfnamefont
  {S.}~\bibnamefont {Chakrapani}}, \bibinfo {author} {\bibfnamefont
  {B.}~\bibnamefont {Roux}}, \ and\ \bibinfo {author} {\bibfnamefont
  {E.}~\bibnamefont {Perozo}},\ }\bibfield  {title} {\enquote {\bibinfo {title}
  {{Structural basis for the coupling between activation and inactivation gates
  in K+ channels}},}\ }\href {\doibase 10.1038/nature09136} {\bibfield
  {journal} {\bibinfo  {journal} {Nature}\ }\textbf {\bibinfo {volume} {466}},\
  \bibinfo {pages} {272--275} (\bibinfo {year} {2010})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Bhate}\ \emph {et~al.}(2010)\citenamefont {Bhate},
  \citenamefont {Wylie}, \citenamefont {Tian},\ and\ \citenamefont
  {McDermott}}]{Bhate2010}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {M.~P.}\ \bibnamefont
  {Bhate}}, \bibinfo {author} {\bibfnamefont {B.~J.}\ \bibnamefont {Wylie}},
  \bibinfo {author} {\bibfnamefont {L.}~\bibnamefont {Tian}}, \ and\ \bibinfo
  {author} {\bibfnamefont {A.~E.}\ \bibnamefont {McDermott}},\ }\bibfield
  {title} {\enquote {\bibinfo {title} {{Conformational dynamics in the
  selectivity filter of KcsA in response to potassium ion concentration}},}\
  }\href {\doibase 10.1016/j.jmb.2010.06.031} {\bibfield  {journal} {\bibinfo
  {journal} {Journal of Molecular Biology}\ }\textbf {\bibinfo {volume}
  {401}},\ \bibinfo {pages} {155--166} (\bibinfo {year} {2010})}\BibitemShut
  {NoStop}%
\bibitem [{\citenamefont {{De Dios}}, \citenamefont {Pearson},\ and\
  \citenamefont {Oldfield}(1993)}]{DeDios1993}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {A.~C.}\ \bibnamefont
  {{De Dios}}}, \bibinfo {author} {\bibfnamefont {J.~G.}\ \bibnamefont
  {Pearson}}, \ and\ \bibinfo {author} {\bibfnamefont {E.}~\bibnamefont
  {Oldfield}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{Secondary and
  tertiary structural effects on protein NMR chemical shifts: An ab initio
  approach}},}\ }\href {\doibase 10.1126/science.8502992} {\bibfield  {journal}
  {\bibinfo  {journal} {Science}\ }\textbf {\bibinfo {volume} {260}},\ \bibinfo
  {pages} {1491--1496} (\bibinfo {year} {1993})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Oldfield}(1995)}]{Oldfield1995}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {E.}~\bibnamefont
  {Oldfield}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{Chemical
  shifts and three-dimensional protein structures}},}\ }\href {\doibase
  10.1007/BF00211749} {\bibfield  {journal} {\bibinfo  {journal} {Journal of
  Biomolecular NMR}\ }\textbf {\bibinfo {volume} {5}},\ \bibinfo {pages}
  {217--225} (\bibinfo {year} {1995})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Wishart}\ and\ \citenamefont
  {Case}(2002)}]{Wishart2002}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {D.~S.}\ \bibnamefont
  {Wishart}}\ and\ \bibinfo {author} {\bibfnamefont {D.~A.}\ \bibnamefont
  {Case}},\ }\href {\doibase 10.1016/S0076-6879(02)38214-4} {\emph {\bibinfo
  {title} {Methods in Enzymology}}},\ Vol.\ \bibinfo {volume} {338}\ (\bibinfo
  {publisher} {Elsevier Masson SAS},\ \bibinfo {year} {2002})\ pp.\ \bibinfo
  {pages} {3--34}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Shen}\ \emph {et~al.}(2008)\citenamefont {Shen},
  \citenamefont {Lange}, \citenamefont {Delaglio}, \citenamefont {Rossi},
  \citenamefont {Aramini}, \citenamefont {Liu}, \citenamefont {Eletsky},
  \citenamefont {Wu}, \citenamefont {Singarapu}, \citenamefont {Lemak},
  \citenamefont {Ignatchenko}, \citenamefont {Arrowsmith}, \citenamefont
  {Szyperski}, \citenamefont {Montelione}, \citenamefont {Baker},\ and\
  \citenamefont {Bax}}]{Shen2008}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {Y.}~\bibnamefont
  {Shen}}, \bibinfo {author} {\bibfnamefont {O.}~\bibnamefont {Lange}},
  \bibinfo {author} {\bibfnamefont {F.}~\bibnamefont {Delaglio}}, \bibinfo
  {author} {\bibfnamefont {P.}~\bibnamefont {Rossi}}, \bibinfo {author}
  {\bibfnamefont {J.~M.}\ \bibnamefont {Aramini}}, \bibinfo {author}
  {\bibfnamefont {G.}~\bibnamefont {Liu}}, \bibinfo {author} {\bibfnamefont
  {A.}~\bibnamefont {Eletsky}}, \bibinfo {author} {\bibfnamefont
  {Y.}~\bibnamefont {Wu}}, \bibinfo {author} {\bibfnamefont {K.~K.}\
  \bibnamefont {Singarapu}}, \bibinfo {author} {\bibfnamefont {A.}~\bibnamefont
  {Lemak}}, \bibinfo {author} {\bibfnamefont {A.}~\bibnamefont {Ignatchenko}},
  \bibinfo {author} {\bibfnamefont {C.~H.}\ \bibnamefont {Arrowsmith}},
  \bibinfo {author} {\bibfnamefont {T.}~\bibnamefont {Szyperski}}, \bibinfo
  {author} {\bibfnamefont {G.~T.}\ \bibnamefont {Montelione}}, \bibinfo
  {author} {\bibfnamefont {D.}~\bibnamefont {Baker}}, \ and\ \bibinfo {author}
  {\bibfnamefont {A.}~\bibnamefont {Bax}},\ }\bibfield  {title} {\enquote
  {\bibinfo {title} {{Consistent blind protein structure generation from NMR
  chemical shift data}},}\ }\href {\doibase 10.1073/pnas.0800256105} {\bibfield
   {journal} {\bibinfo  {journal} {Proceedings of the National Academy of
  Sciences of the United States of America}\ }\textbf {\bibinfo {volume}
  {105}},\ \bibinfo {pages} {4685--4690} (\bibinfo {year} {2008})}\BibitemShut
  {NoStop}%
\bibitem [{\citenamefont {Cavalli}\ \emph {et~al.}(2007)\citenamefont
  {Cavalli}, \citenamefont {Salvatella}, \citenamefont {Dobson},\ and\
  \citenamefont {Vendruscolo}}]{Cavalli2007}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {A.}~\bibnamefont
  {Cavalli}}, \bibinfo {author} {\bibfnamefont {X.}~\bibnamefont {Salvatella}},
  \bibinfo {author} {\bibfnamefont {C.~M.}\ \bibnamefont {Dobson}}, \ and\
  \bibinfo {author} {\bibfnamefont {M.}~\bibnamefont {Vendruscolo}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Protein structure
  determination from NMR chemical shifts}},}\ }\href {\doibase
  10.1073/pnas.0610313104} {\bibfield  {journal} {\bibinfo  {journal}
  {Proceedings of the National Academy of Sciences of the United States of
  America}\ }\textbf {\bibinfo {volume} {104}},\ \bibinfo {pages} {9615--9620}
  (\bibinfo {year} {2007})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Niklasson}\ \emph {et~al.}(2015)\citenamefont
  {Niklasson}, \citenamefont {Ahlner}, \citenamefont {Andresen}, \citenamefont
  {Marsh},\ and\ \citenamefont {Lundstr{\"{o}}m}}]{Niklasson2015}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {M.}~\bibnamefont
  {Niklasson}}, \bibinfo {author} {\bibfnamefont {A.}~\bibnamefont {Ahlner}},
  \bibinfo {author} {\bibfnamefont {C.}~\bibnamefont {Andresen}}, \bibinfo
  {author} {\bibfnamefont {J.~A.}\ \bibnamefont {Marsh}}, \ and\ \bibinfo
  {author} {\bibfnamefont {P.}~\bibnamefont {Lundstr{\"{o}}m}},\ }\bibfield
  {title} {\enquote {\bibinfo {title} {{Fast and Accurate Resonance Assignment
  of Small-to-Large Proteins by Combining Automated and Manual Approaches}},}\
  }\href {\doibase 10.1371/journal.pcbi.1004022} {\bibfield  {journal}
  {\bibinfo  {journal} {PLoS Computational Biology}\ }\textbf {\bibinfo
  {volume} {11}} (\bibinfo {year} {2015}),\
  10.1371/journal.pcbi.1004022}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Berjanskii}\ \emph {et~al.}(2015)\citenamefont
  {Berjanskii}, \citenamefont {Arndt}, \citenamefont {Liang},\ and\
  \citenamefont {Wishart}}]{Berjanskii2015}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {M.}~\bibnamefont
  {Berjanskii}}, \bibinfo {author} {\bibfnamefont {D.}~\bibnamefont {Arndt}},
  \bibinfo {author} {\bibfnamefont {Y.}~\bibnamefont {Liang}}, \ and\ \bibinfo
  {author} {\bibfnamefont {D.~S.}\ \bibnamefont {Wishart}},\ }\bibfield
  {title} {\enquote {\bibinfo {title} {{A robust algorithm for optimizing
  protein structures with NMR chemical shifts}},}\ }\href {\doibase
  10.1007/s10858-015-9982-z} {\bibfield  {journal} {\bibinfo  {journal}
  {Journal of Biomolecular NMR}\ }\textbf {\bibinfo {volume} {63}},\ \bibinfo
  {pages} {255--264} (\bibinfo {year} {2015})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Camilloni}\ \emph {et~al.}(2012)\citenamefont
  {Camilloni}, \citenamefont {Robustelli}, \citenamefont {Simone},
  \citenamefont {Cavalli},\ and\ \citenamefont
  {Vendruscolo}}]{camilloni2012characterization}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {C.}~\bibnamefont
  {Camilloni}}, \bibinfo {author} {\bibfnamefont {P.}~\bibnamefont
  {Robustelli}}, \bibinfo {author} {\bibfnamefont {A.~D.}\ \bibnamefont
  {Simone}}, \bibinfo {author} {\bibfnamefont {A.}~\bibnamefont {Cavalli}}, \
  and\ \bibinfo {author} {\bibfnamefont {M.}~\bibnamefont {Vendruscolo}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {Characterization of the
  conformational equilibrium between the two major substates of rnase a using
  nmr chemical shifts},}\ }\href@noop {} {\bibfield  {journal} {\bibinfo
  {journal} {Journal of the American Chemical Society}\ }\textbf {\bibinfo
  {volume} {134}},\ \bibinfo {pages} {3968--3971} (\bibinfo {year}
  {2012})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Robustelli}, \citenamefont {Stafford},\ and\
  \citenamefont {Palmer}(2012)}]{Robustelli2012}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {P.}~\bibnamefont
  {Robustelli}}, \bibinfo {author} {\bibfnamefont {K.~A.}\ \bibnamefont
  {Stafford}}, \ and\ \bibinfo {author} {\bibfnamefont {A.~G.}\ \bibnamefont
  {Palmer}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{Interpreting
  protein structural dynamics from NMR chemical shifts}},}\ }\href {\doibase
  10.1021/ja300265w} {\bibfield  {journal} {\bibinfo  {journal} {Journal of the
  American Chemical Society}\ }\textbf {\bibinfo {volume} {134}},\ \bibinfo
  {pages} {6365--6374} (\bibinfo {year} {2012})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Markwick}\ \emph {et~al.}(2010)\citenamefont
  {Markwick}, \citenamefont {Cervantes}, \citenamefont {Abel}, \citenamefont
  {Komives}, \citenamefont {Blackledge},\ and\ \citenamefont
  {McCammon}}]{Markwick2010}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {P.~R.}\ \bibnamefont
  {Markwick}}, \bibinfo {author} {\bibfnamefont {C.~F.}\ \bibnamefont
  {Cervantes}}, \bibinfo {author} {\bibfnamefont {B.~L.}\ \bibnamefont {Abel}},
  \bibinfo {author} {\bibfnamefont {E.~A.}\ \bibnamefont {Komives}}, \bibinfo
  {author} {\bibfnamefont {M.}~\bibnamefont {Blackledge}}, \ and\ \bibinfo
  {author} {\bibfnamefont {J.~A.}\ \bibnamefont {McCammon}},\ }\bibfield
  {title} {\enquote {\bibinfo {title} {{Enhanced conformational space sampling
  improves the prediction of chemical shifts in proteins}},}\ }\href {\doibase
  10.1021/ja9093692} {\bibfield  {journal} {\bibinfo  {journal} {Journal of the
  American Chemical Society}\ }\textbf {\bibinfo {volume} {132}},\ \bibinfo
  {pages} {1220--1221} (\bibinfo {year} {2010})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Li}\ and\ \citenamefont
  {Br{\"{u}}schweiler}(2010)}]{Li2010}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {D.~W.}\ \bibnamefont
  {Li}}\ and\ \bibinfo {author} {\bibfnamefont {R.}~\bibnamefont
  {Br{\"{u}}schweiler}},\ }\bibfield  {title} {\enquote {\bibinfo {title}
  {{Certification of molecular dynamics trajectories with NMR chemical
  shifts}},}\ }\href {\doibase 10.1021/jz9001345} {\bibfield  {journal}
  {\bibinfo  {journal} {Journal of Physical Chemistry Letters}\ }\textbf
  {\bibinfo {volume} {1}},\ \bibinfo {pages} {246--248} (\bibinfo {year}
  {2010})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Li}\ and\ \citenamefont
  {Br{\"{u}}schweiler}(2012)}]{Li2012}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {D.~W.}\ \bibnamefont
  {Li}}\ and\ \bibinfo {author} {\bibfnamefont {R.}~\bibnamefont
  {Br{\"{u}}schweiler}},\ }\bibfield  {title} {\enquote {\bibinfo {title}
  {{PPM: A side-chain and backbone chemical shift predictor for the assessment
  of protein conformational ensembles}},}\ }\href {\doibase
  10.1007/s10858-012-9668-8} {\bibfield  {journal} {\bibinfo  {journal}
  {Journal of Biomolecular NMR}\ }\textbf {\bibinfo {volume} {54}},\ \bibinfo
  {pages} {257--265} (\bibinfo {year} {2012})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Li}\ and\ \citenamefont
  {Br{\"{u}}schweiler}(2015)}]{Li2015}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {D.}~\bibnamefont
  {Li}}\ and\ \bibinfo {author} {\bibfnamefont {R.}~\bibnamefont
  {Br{\"{u}}schweiler}},\ }\bibfield  {title} {\enquote {\bibinfo {title}
  {{PPM-One: A static protein structure based chemical shift predictor}},}\
  }\href {\doibase 10.1007/s10858-015-9958-z} {\bibfield  {journal} {\bibinfo
  {journal} {Journal of Biomolecular NMR}\ }\textbf {\bibinfo {volume} {62}},\
  \bibinfo {pages} {403--409} (\bibinfo {year} {2015})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Zhang}, \citenamefont {Itin},\ and\ \citenamefont
  {McDermott}(2019)}]{Zhang2019}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {D.}~\bibnamefont
  {Zhang}}, \bibinfo {author} {\bibfnamefont {B.}~\bibnamefont {Itin}}, \ and\
  \bibinfo {author} {\bibfnamefont {A.~E.}\ \bibnamefont {McDermott}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{TmDOTP: An NMR-based
  thermometer for magic angle spinning NMR experiments}},}\ }\href {\doibase
  10.1016/j.jmr.2019.106574} {\bibfield  {journal} {\bibinfo  {journal}
  {Journal of Magnetic Resonance}\ }\textbf {\bibinfo {volume} {308}},\
  \bibinfo {pages} {106574} (\bibinfo {year} {2019})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Pauli}\ \emph {et~al.}(2001)\citenamefont {Pauli},
  \citenamefont {Baldus}, \citenamefont {{Van Rossum}}, \citenamefont {{De
  Groot}},\ and\ \citenamefont {Oschkinat}}]{Pauli2001}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {J.}~\bibnamefont
  {Pauli}}, \bibinfo {author} {\bibfnamefont {M.}~\bibnamefont {Baldus}},
  \bibinfo {author} {\bibfnamefont {B.}~\bibnamefont {{Van Rossum}}}, \bibinfo
  {author} {\bibfnamefont {H.}~\bibnamefont {{De Groot}}}, \ and\ \bibinfo
  {author} {\bibfnamefont {H.}~\bibnamefont {Oschkinat}},\ }\bibfield  {title}
  {\enquote {\bibinfo {title} {{Backbone and side-chain \ce{^13C} and \ce{^15N}
  signal assignments of the $\alpha$-spectrin SH3 domain by magic angle
  spinning solid-state NMR at 17.6 tesla}},}\ }\href {\doibase
  10.1002/1439-7633(20010401)2:4<272::AID-CBIC272>3.0.CO;2-2} {\bibfield
  {journal} {\bibinfo  {journal} {ChemBioChem}\ }\textbf {\bibinfo {volume}
  {2}},\ \bibinfo {pages} {272--281} (\bibinfo {year} {2001})}\BibitemShut
  {NoStop}%
\bibitem [{\citenamefont {Igumenova}, \citenamefont {Wand},\ and\ \citenamefont
  {McDermott}(2004)}]{Igumenova2004}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {T.~I.}\ \bibnamefont
  {Igumenova}}, \bibinfo {author} {\bibfnamefont {A.~J.}\ \bibnamefont {Wand}},
  \ and\ \bibinfo {author} {\bibfnamefont {A.~E.}\ \bibnamefont {McDermott}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Assignment of the Backbone
  Resonances for Microcrystalline Ubiquitin}},}\ }\href {\doibase
  10.1021/ja030546w} {\bibfield  {journal} {\bibinfo  {journal} {Journal of the
  American Chemical Society}\ }\textbf {\bibinfo {volume} {126}},\ \bibinfo
  {pages} {5323--5331} (\bibinfo {year} {2004})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Li}\ \emph {et~al.}(2007)\citenamefont {Li},
  \citenamefont {Berthold}, \citenamefont {Frericks}, \citenamefont {Gennis},\
  and\ \citenamefont {Rienstra}}]{Li2007}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {Y.}~\bibnamefont
  {Li}}, \bibinfo {author} {\bibfnamefont {D.~A.}\ \bibnamefont {Berthold}},
  \bibinfo {author} {\bibfnamefont {H.~L.}\ \bibnamefont {Frericks}}, \bibinfo
  {author} {\bibfnamefont {R.~B.}\ \bibnamefont {Gennis}}, \ and\ \bibinfo
  {author} {\bibfnamefont {C.~M.}\ \bibnamefont {Rienstra}},\ }\bibfield
  {title} {\enquote {\bibinfo {title} {{Partial \ce{^13C} and \ce{^15N}
  chemical-shift assignments of the disulfide-bond-forming enzyme DsbB by 3D
  magic-angle spinning NMR spectroscopy}},}\ }\href {\doibase
  10.1002/cbic.200600484} {\bibfield  {journal} {\bibinfo  {journal}
  {ChemBioChem}\ }\textbf {\bibinfo {volume} {8}},\ \bibinfo {pages} {434--442}
  (\bibinfo {year} {2007})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Takegoshi}, \citenamefont {Nakamura},\ and\
  \citenamefont {Terao}(2001)}]{Takegoshi2001}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {K.}~\bibnamefont
  {Takegoshi}}, \bibinfo {author} {\bibfnamefont {S.}~\bibnamefont {Nakamura}},
  \ and\ \bibinfo {author} {\bibfnamefont {T.}~\bibnamefont {Terao}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{\ce{^13C}-\ce{^1H}
  dipolar-assisted rotational resonance in magic-angle spinning NMR}},}\ }\href
  {\doibase 10.1021/ja016789a} {\bibfield  {journal} {\bibinfo  {journal}
  {Chemical Physics Letters}\ }\textbf {\bibinfo {volume} {344}},\ \bibinfo
  {pages} {631--637} (\bibinfo {year} {2001})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Baldus}\ \emph {et~al.}(1998)\citenamefont {Baldus},
  \citenamefont {Petkova}, \citenamefont {Herzfeld},\ and\ \citenamefont
  {Griffin}}]{Baldus1998}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {M.}~\bibnamefont
  {Baldus}}, \bibinfo {author} {\bibfnamefont {A.~T.}\ \bibnamefont {Petkova}},
  \bibinfo {author} {\bibfnamefont {J.}~\bibnamefont {Herzfeld}}, \ and\
  \bibinfo {author} {\bibfnamefont {R.~G.}\ \bibnamefont {Griffin}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Cross polarization in the
  tilted frame: Assignment and spectral simplification in heteronuclear spin
  systems}},}\ }\href {\doibase 10.1080/00268979809483251} {\bibfield
  {journal} {\bibinfo  {journal} {Molecular Physics}\ }\textbf {\bibinfo
  {volume} {95}},\ \bibinfo {pages} {1197--1207} (\bibinfo {year}
  {1998})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Fung}, \citenamefont {Khitrin},\ and\ \citenamefont
  {Ermolaev}(2000)}]{Fung2000}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {B.~M.}\ \bibnamefont
  {Fung}}, \bibinfo {author} {\bibfnamefont {A.~K.}\ \bibnamefont {Khitrin}}, \
  and\ \bibinfo {author} {\bibfnamefont {K.}~\bibnamefont {Ermolaev}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{An Improved Broadband
  Decoupling Sequence for Liquid Crystals and Solids}},}\ }\href {\doibase
  10.1006/jmre.1999.1896} {\bibfield  {journal} {\bibinfo  {journal} {Journal
  of Magnetic Resonance}\ }\textbf {\bibinfo {volume} {142}},\ \bibinfo {pages}
  {97--101} (\bibinfo {year} {2000})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Hou}\ \emph {et~al.}(2013)\citenamefont {Hou},
  \citenamefont {Yan}, \citenamefont {Tr{\'{e}}bosc}, \citenamefont
  {Amoureux},\ and\ \citenamefont {Polenova}}]{Hou2013}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {G.}~\bibnamefont
  {Hou}}, \bibinfo {author} {\bibfnamefont {S.}~\bibnamefont {Yan}}, \bibinfo
  {author} {\bibfnamefont {J.}~\bibnamefont {Tr{\'{e}}bosc}}, \bibinfo {author}
  {\bibfnamefont {J.~P.}\ \bibnamefont {Amoureux}}, \ and\ \bibinfo {author}
  {\bibfnamefont {T.}~\bibnamefont {Polenova}},\ }\bibfield  {title} {\enquote
  {\bibinfo {title} {{Broadband homonuclear correlation spectroscopy driven by
  combined $R2^v_n$ sequences under fast magic angle spinning for NMR
  structural analysis of organic and biological solids}},}\ }\href {\doibase
  10.1016/j.jmr.2013.04.009} {\bibfield  {journal} {\bibinfo  {journal}
  {Journal of Magnetic Resonance}\ }\textbf {\bibinfo {volume} {232}},\
  \bibinfo {pages} {18--30} (\bibinfo {year} {2013})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Jaroniec}, \citenamefont {Filip},\ and\ \citenamefont
  {Griffin}(2002)}]{Jaroniec2002}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {C.~P.}\ \bibnamefont
  {Jaroniec}}, \bibinfo {author} {\bibfnamefont {C.}~\bibnamefont {Filip}}, \
  and\ \bibinfo {author} {\bibfnamefont {R.~G.}\ \bibnamefont {Griffin}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{3D TEDOR NMR experiments
  for the simultaneous measurement of multiple carbon-nitrogen distances in
  uniformly \ce{^13C},\ce{^15N}-labeled solids}},}\ }\href {\doibase
  10.1021/ja026385y} {\bibfield  {journal} {\bibinfo  {journal} {Journal of the
  American Chemical Society}\ }\textbf {\bibinfo {volume} {124}},\ \bibinfo
  {pages} {10728--10742} (\bibinfo {year} {2002})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Thakur}, \citenamefont {Kurur},\ and\ \citenamefont
  {Madhu}(2006)}]{Thakur2006}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {R.~S.}\ \bibnamefont
  {Thakur}}, \bibinfo {author} {\bibfnamefont {N.~D.}\ \bibnamefont {Kurur}}, \
  and\ \bibinfo {author} {\bibfnamefont {P.~K.}\ \bibnamefont {Madhu}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Swept-frequency two-pulse
  phase modulation for heteronuclear dipolar decoupling in solid-state NMR}},}\
  }\href {\doibase 10.1016/j.cplett.2006.06.007} {\bibfield  {journal}
  {\bibinfo  {journal} {Chemical Physics Letters}\ }\textbf {\bibinfo {volume}
  {426}},\ \bibinfo {pages} {459--463} (\bibinfo {year} {2006})}\BibitemShut
  {NoStop}%
\bibitem [{\citenamefont {Delaglio}\ \emph {et~al.}(1995)\citenamefont
  {Delaglio}, \citenamefont {Grzesiek}, \citenamefont {Vuister}, \citenamefont
  {Zhu}, \citenamefont {Pfeifer},\ and\ \citenamefont {Bax}}]{Delaglio1995}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {F.}~\bibnamefont
  {Delaglio}}, \bibinfo {author} {\bibfnamefont {S.}~\bibnamefont {Grzesiek}},
  \bibinfo {author} {\bibfnamefont {G.~W.}\ \bibnamefont {Vuister}}, \bibinfo
  {author} {\bibfnamefont {G.}~\bibnamefont {Zhu}}, \bibinfo {author}
  {\bibfnamefont {J.}~\bibnamefont {Pfeifer}}, \ and\ \bibinfo {author}
  {\bibfnamefont {A.}~\bibnamefont {Bax}},\ }\bibfield  {title} {\enquote
  {\bibinfo {title} {{NMRPipe: A multidimensional spectral processing system
  based on UNIX pipes}},}\ }\href {\doibase 10.1007/BF00197809} {\bibfield
  {journal} {\bibinfo  {journal} {Journal of Biomolecular NMR}\ }\textbf
  {\bibinfo {volume} {6}},\ \bibinfo {pages} {277--293} (\bibinfo {year}
  {1995})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Morcombe}\ and\ \citenamefont
  {Zilm}(2003)}]{Morcombe2003}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {C.~R.}\ \bibnamefont
  {Morcombe}}\ and\ \bibinfo {author} {\bibfnamefont {K.~W.}\ \bibnamefont
  {Zilm}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{Chemical shift
  referencing in MAS solid state NMR}},}\ }\href {\doibase
  10.1016/S1090-7807(03)00082-X} {\bibfield  {journal} {\bibinfo  {journal}
  {Journal of Magnetic Resonance}\ }\textbf {\bibinfo {volume} {162}},\
  \bibinfo {pages} {479--486} (\bibinfo {year} {2003})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {McDermott}\ and\ \citenamefont
  {Gu}(2007)}]{McDermott2007}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {A.}~\bibnamefont
  {McDermott}}\ and\ \bibinfo {author} {\bibfnamefont {Z.}~\bibnamefont {Gu}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Carbon and Nitrogen
  Chemical Shifts of Solid State Enzymes}},}\ }\href {\doibase
  10.1002/9780470034590.emrstm0053} {\bibfield  {journal} {\bibinfo  {journal}
  {Encyclopedia of Magnetic Resonance}\ } (\bibinfo {year} {2007}),\
  10.1002/9780470034590.emrstm0053}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Harris}\ \emph {et~al.}(2008)\citenamefont {Harris},
  \citenamefont {Becker}, \citenamefont {{De Cabral Menezes}}, \citenamefont
  {Granger}, \citenamefont {Hoffman},\ and\ \citenamefont {Zilm}}]{Harris2008}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {R.~K.}\ \bibnamefont
  {Harris}}, \bibinfo {author} {\bibfnamefont {E.~D.}\ \bibnamefont {Becker}},
  \bibinfo {author} {\bibfnamefont {S.~M.}\ \bibnamefont {{De Cabral
  Menezes}}}, \bibinfo {author} {\bibfnamefont {P.}~\bibnamefont {Granger}},
  \bibinfo {author} {\bibfnamefont {R.~E.}\ \bibnamefont {Hoffman}}, \ and\
  \bibinfo {author} {\bibfnamefont {K.~W.}\ \bibnamefont {Zilm}},\ }\bibfield
  {title} {\enquote {\bibinfo {title} {{Further Conventions for NMR shielding
  and chemical shifts (IUPAC recommendations 2008)}},}\ }\href {\doibase
  10.1002/mrc.2225} {\bibfield  {journal} {\bibinfo  {journal} {Magnetic
  Resonance in Chemistry}\ }\textbf {\bibinfo {volume} {46}},\ \bibinfo {pages}
  {582--598} (\bibinfo {year} {2008})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Bertani}, \citenamefont {Raya},\ and\ \citenamefont
  {Bechinger}(2014)}]{Bertani2014}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {P.}~\bibnamefont
  {Bertani}}, \bibinfo {author} {\bibfnamefont {J.}~\bibnamefont {Raya}}, \
  and\ \bibinfo {author} {\bibfnamefont {B.}~\bibnamefont {Bechinger}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{\ce{^15N} chemical shift
  referencing in solid state NMR}},}\ }\href {\doibase
  10.1016/j.ssnmr.2014.03.003} {\bibfield  {journal} {\bibinfo  {journal}
  {Solid State Nuclear Magnetic Resonance}\ }\textbf {\bibinfo {volume}
  {61-62}},\ \bibinfo {pages} {15--18} (\bibinfo {year} {2014})}\BibitemShut
  {NoStop}%
\bibitem [{\citenamefont {Vranken}\ \emph {et~al.}(2005)\citenamefont
  {Vranken}, \citenamefont {Boucher}, \citenamefont {Stevens}, \citenamefont
  {Fogh}, \citenamefont {Pajon}, \citenamefont {Llinas}, \citenamefont
  {Ulrich}, \citenamefont {Markley}, \citenamefont {Ionides},\ and\
  \citenamefont {Laue}}]{Vranken2005}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {W.~F.}\ \bibnamefont
  {Vranken}}, \bibinfo {author} {\bibfnamefont {W.}~\bibnamefont {Boucher}},
  \bibinfo {author} {\bibfnamefont {T.~J.}\ \bibnamefont {Stevens}}, \bibinfo
  {author} {\bibfnamefont {R.~H.}\ \bibnamefont {Fogh}}, \bibinfo {author}
  {\bibfnamefont {A.}~\bibnamefont {Pajon}}, \bibinfo {author} {\bibfnamefont
  {M.}~\bibnamefont {Llinas}}, \bibinfo {author} {\bibfnamefont {E.~L.}\
  \bibnamefont {Ulrich}}, \bibinfo {author} {\bibfnamefont {J.~L.}\
  \bibnamefont {Markley}}, \bibinfo {author} {\bibfnamefont {J.}~\bibnamefont
  {Ionides}}, \ and\ \bibinfo {author} {\bibfnamefont {E.~D.}\ \bibnamefont
  {Laue}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{The CCPN data
  model for NMR spectroscopy: Development of a software pipeline}},}\ }\href
  {\doibase 10.1002/prot.20449} {\bibfield  {journal} {\bibinfo  {journal}
  {Proteins: Structure, Function and Genetics}\ }\textbf {\bibinfo {volume}
  {59}},\ \bibinfo {pages} {687--696} (\bibinfo {year} {2005})}\BibitemShut
  {NoStop}%
\bibitem [{\citenamefont {Fritzsching}, \citenamefont {Hong},\ and\
  \citenamefont {Schmidt-Rohr}(2016)}]{Fritzsching2016}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {K.~J.}\ \bibnamefont
  {Fritzsching}}, \bibinfo {author} {\bibfnamefont {M.}~\bibnamefont {Hong}}, \
  and\ \bibinfo {author} {\bibfnamefont {K.}~\bibnamefont {Schmidt-Rohr}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Conformationally selective
  multidimensional chemical shift ranges in proteins from a PACSY database
  purged using intrinsic quality criteria}},}\ }\href {\doibase
  10.1007/s10858-016-0013-5} {\bibfield  {journal} {\bibinfo  {journal}
  {Journal of Biomolecular NMR}\ }\textbf {\bibinfo {volume} {64}},\ \bibinfo
  {pages} {115--130} (\bibinfo {year} {2016})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Yang}, \citenamefont {Fritzsching},\ and\
  \citenamefont {Hong}(2013)}]{Yang2013}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {Y.}~\bibnamefont
  {Yang}}, \bibinfo {author} {\bibfnamefont {K.~J.}\ \bibnamefont
  {Fritzsching}}, \ and\ \bibinfo {author} {\bibfnamefont {M.}~\bibnamefont
  {Hong}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{Resonance
  assignment of the NMR spectra of disordered proteins using a multi-objective
  non-dominated sorting genetic algorithm}},}\ }\href {\doibase
  10.1007/s10858-013-9788-9} {\bibfield  {journal} {\bibinfo  {journal}
  {Journal of Biomolecular NMR}\ }\textbf {\bibinfo {volume} {57}},\ \bibinfo
  {pages} {281--296} (\bibinfo {year} {2013})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Jo}\ \emph {et~al.}(2017)\citenamefont {Jo},
  \citenamefont {Cheng}, \citenamefont {Lee}, \citenamefont {Kim},
  \citenamefont {Park}, \citenamefont {Patel}, \citenamefont {Beaven},
  \citenamefont {Lee}, \citenamefont {Rui}, \citenamefont {Park}, \citenamefont
  {Lee}, \citenamefont {Roux}, \citenamefont {MacKerell}, \citenamefont
  {Klauda}, \citenamefont {Qi},\ and\ \citenamefont {Im}}]{Jo2017}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {S.}~\bibnamefont
  {Jo}}, \bibinfo {author} {\bibfnamefont {X.}~\bibnamefont {Cheng}}, \bibinfo
  {author} {\bibfnamefont {J.}~\bibnamefont {Lee}}, \bibinfo {author}
  {\bibfnamefont {S.}~\bibnamefont {Kim}}, \bibinfo {author} {\bibfnamefont
  {S.~J.}\ \bibnamefont {Park}}, \bibinfo {author} {\bibfnamefont {D.~S.}\
  \bibnamefont {Patel}}, \bibinfo {author} {\bibfnamefont {A.~H.}\ \bibnamefont
  {Beaven}}, \bibinfo {author} {\bibfnamefont {K.~I.}\ \bibnamefont {Lee}},
  \bibinfo {author} {\bibfnamefont {H.}~\bibnamefont {Rui}}, \bibinfo {author}
  {\bibfnamefont {S.}~\bibnamefont {Park}}, \bibinfo {author} {\bibfnamefont
  {H.~S.}\ \bibnamefont {Lee}}, \bibinfo {author} {\bibfnamefont
  {B.}~\bibnamefont {Roux}}, \bibinfo {author} {\bibfnamefont {A.~D.}\
  \bibnamefont {MacKerell}}, \bibinfo {author} {\bibfnamefont {J.~B.}\
  \bibnamefont {Klauda}}, \bibinfo {author} {\bibfnamefont {Y.}~\bibnamefont
  {Qi}}, \ and\ \bibinfo {author} {\bibfnamefont {W.}~\bibnamefont {Im}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{CHARMM-GUI 10 years for
  biomolecular modeling and simulation}},}\ }\href {\doibase 10.1002/jcc.24660}
  {\bibfield  {journal} {\bibinfo  {journal} {J. Comput. Chem.}\ }\textbf
  {\bibinfo {volume} {38}},\ \bibinfo {pages} {1114--1124} (\bibinfo {year}
  {2017})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {DiMaio}\ \emph {et~al.}(2011)\citenamefont {DiMaio},
  \citenamefont {Leaver-Fay}, \citenamefont {Bradley}, \citenamefont {Baker},\
  and\ \citenamefont {Andr{\'{e}}}}]{DiMaio2011}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {F.}~\bibnamefont
  {DiMaio}}, \bibinfo {author} {\bibfnamefont {A.}~\bibnamefont {Leaver-Fay}},
  \bibinfo {author} {\bibfnamefont {P.}~\bibnamefont {Bradley}}, \bibinfo
  {author} {\bibfnamefont {D.}~\bibnamefont {Baker}}, \ and\ \bibinfo {author}
  {\bibfnamefont {I.}~\bibnamefont {Andr{\'{e}}}},\ }\bibfield  {title}
  {\enquote {\bibinfo {title} {{Modeling symmetric Macromolecular structures in
  Rosetta3}},}\ }\href {\doibase 10.1371/journal.pone.0020450} {\bibfield
  {journal} {\bibinfo  {journal} {PLoS One}\ }\textbf {\bibinfo {volume} {6}}
  (\bibinfo {year} {2011}),\ 10.1371/journal.pone.0020450}\BibitemShut
  {NoStop}%
\bibitem [{\citenamefont {{Schr\"odinger, LLC}}(2015)}]{PyMOL}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibnamefont {{Schr\"odinger,
  LLC}}},\ }\href@noop {} {\enquote {\bibinfo {title} {The {PyMOL} molecular
  graphics system, version~1.8},}\ } (\bibinfo {year} {2015})\BibitemShut
  {NoStop}%
\bibitem [{\citenamefont {Bhate}\ and\ \citenamefont
  {McDermott}(2012)}]{Bhate2012}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {M.~P.}\ \bibnamefont
  {Bhate}}\ and\ \bibinfo {author} {\bibfnamefont {A.~E.}\ \bibnamefont
  {McDermott}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{Protonation
  state of E71 in KcsA and its role for channel collapse and inactivation}},}\
  }\href {\doibase 10.1073/pnas.1211900109} {\bibfield  {journal} {\bibinfo
  {journal} {Proceedings of the National Academy of Sciences of the United
  States of America}\ }\textbf {\bibinfo {volume} {109}},\ \bibinfo {pages}
  {15265--15270} (\bibinfo {year} {2012})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Jorgensen}\ \emph {et~al.}(1983)\citenamefont
  {Jorgensen}, \citenamefont {Chandrasekhar}, \citenamefont {Madura},
  \citenamefont {Impey},\ and\ \citenamefont
  {Klein}}]{TIP3P_JChemPhys_Jorgensen_1983}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {W.~L.}\ \bibnamefont
  {Jorgensen}}, \bibinfo {author} {\bibfnamefont {J.}~\bibnamefont
  {Chandrasekhar}}, \bibinfo {author} {\bibfnamefont {J.~D.}\ \bibnamefont
  {Madura}}, \bibinfo {author} {\bibfnamefont {R.~W.}\ \bibnamefont {Impey}}, \
  and\ \bibinfo {author} {\bibfnamefont {M.~L.}\ \bibnamefont {Klein}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Comparison of simple
  potential functions for simulating liquid water}},}\ }\href@noop {}
  {\bibfield  {journal} {\bibinfo  {journal} {J. Chem. Phys.}\ }\textbf
  {\bibinfo {volume} {79}},\ \bibinfo {pages} {926--935} (\bibinfo {year}
  {1983})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Best}\ \emph {et~al.}(2012)\citenamefont {Best},
  \citenamefont {Zhu}, \citenamefont {Shim}, \citenamefont {Lopes},
  \citenamefont {Mittal}, \citenamefont {Feig},\ and\ \citenamefont
  {MacKerell}}]{Best2012}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {R.~B.}\ \bibnamefont
  {Best}}, \bibinfo {author} {\bibfnamefont {X.}~\bibnamefont {Zhu}}, \bibinfo
  {author} {\bibfnamefont {J.}~\bibnamefont {Shim}}, \bibinfo {author}
  {\bibfnamefont {P.~E.}\ \bibnamefont {Lopes}}, \bibinfo {author}
  {\bibfnamefont {J.}~\bibnamefont {Mittal}}, \bibinfo {author} {\bibfnamefont
  {M.}~\bibnamefont {Feig}}, \ and\ \bibinfo {author} {\bibfnamefont {A.~D.}\
  \bibnamefont {MacKerell}},\ }\bibfield  {title} {\enquote {\bibinfo {title}
  {{Optimization of the additive CHARMM all-atom protein force field targeting
  improved sampling of the backbone $\phi$, $\psi$ and side-chain $\chi$1 and
  $\chi$2 Dihedral Angles}},}\ }\href {\doibase 10.1021/ct300400x} {\bibfield
  {journal} {\bibinfo  {journal} {J. Chem. Theory Comput.}\ }\textbf {\bibinfo
  {volume} {8}},\ \bibinfo {pages} {3257--3273} (\bibinfo {year}
  {2012})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Abraham}\ \emph {et~al.}(2015)\citenamefont
  {Abraham}, \citenamefont {Murtola}, \citenamefont {Schulz}, \citenamefont
  {P{\'{a}}ll}, \citenamefont {Smith}, \citenamefont {Hess},\ and\
  \citenamefont {Lindah}}]{GROMACS_Abraham2015}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {M.~J.}\ \bibnamefont
  {Abraham}}, \bibinfo {author} {\bibfnamefont {T.}~\bibnamefont {Murtola}},
  \bibinfo {author} {\bibfnamefont {R.}~\bibnamefont {Schulz}}, \bibinfo
  {author} {\bibfnamefont {S.}~\bibnamefont {P{\'{a}}ll}}, \bibinfo {author}
  {\bibfnamefont {J.~C.}\ \bibnamefont {Smith}}, \bibinfo {author}
  {\bibfnamefont {B.}~\bibnamefont {Hess}}, \ and\ \bibinfo {author}
  {\bibfnamefont {E.}~\bibnamefont {Lindah}},\ }\bibfield  {title} {\enquote
  {\bibinfo {title} {{Gromacs: High performance molecular simulations through
  multi-level parallelism from laptops to supercomputers}},}\ }\href {\doibase
  10.1016/j.softx.2015.06.001} {\bibfield  {journal} {\bibinfo  {journal}
  {SoftwareX}\ }\textbf {\bibinfo {volume} {1-2}},\ \bibinfo {pages} {19--25}
  (\bibinfo {year} {2015})},\ \Eprint {http://arxiv.org/abs/arXiv:1503.05249v1}
  {arXiv:arXiv:1503.05249v1} \BibitemShut {NoStop}%
\bibitem [{\citenamefont {Parrinello}\ and\ \citenamefont
  {Rahman}(1981)}]{Parrinello1981}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {M.}~\bibnamefont
  {Parrinello}}\ and\ \bibinfo {author} {\bibfnamefont {A.}~\bibnamefont
  {Rahman}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{Polymorphic
  transitions in single crystals: A new molecular dynamics method}},}\ }\href
  {\doibase 10.1063/1.328693} {\bibfield  {journal} {\bibinfo  {journal} {J.
  Appl. Phys.}\ }\textbf {\bibinfo {volume} {52}},\ \bibinfo {pages}
  {7182--7190} (\bibinfo {year} {1981})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Bussi}, \citenamefont {Donadio},\ and\ \citenamefont
  {Parrinello}(2007)}]{Bussi2007}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {G.}~\bibnamefont
  {Bussi}}, \bibinfo {author} {\bibfnamefont {D.}~\bibnamefont {Donadio}}, \
  and\ \bibinfo {author} {\bibfnamefont {M.}~\bibnamefont {Parrinello}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Canonical sampling through
  velocity rescaling}},}\ }\href {\doibase 10.1063/1.2408420} {\bibfield
  {journal} {\bibinfo  {journal} {J. Chem. Phys.}\ }\textbf {\bibinfo {volume}
  {126}} (\bibinfo {year} {2007}),\ 10.1063/1.2408420},\ \Eprint
  {http://arxiv.org/abs/arXiv:0803.4060v1} {arXiv:arXiv:0803.4060v1}
  \BibitemShut {NoStop}%
\bibitem [{\citenamefont {Shen}\ and\ \citenamefont {Bax}(2010)}]{Shen2010}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {Y.}~\bibnamefont
  {Shen}}\ and\ \bibinfo {author} {\bibfnamefont {A.}~\bibnamefont {Bax}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{SPARTA+: A modest
  improvement in empirical NMR chemical shift prediction by means of an
  artificial neural network}},}\ }\href {\doibase 10.1007/s10858-010-9433-9}
  {\bibfield  {journal} {\bibinfo  {journal} {Journal of Biomolecular NMR}\
  }\textbf {\bibinfo {volume} {48}},\ \bibinfo {pages} {13--22} (\bibinfo
  {year} {2010})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Han}\ \emph {et~al.}(2011)\citenamefont {Han},
  \citenamefont {Liu}, \citenamefont {Ginzinger},\ and\ \citenamefont
  {Wishart}}]{Han2011}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {B.}~\bibnamefont
  {Han}}, \bibinfo {author} {\bibfnamefont {Y.}~\bibnamefont {Liu}}, \bibinfo
  {author} {\bibfnamefont {S.~W.}\ \bibnamefont {Ginzinger}}, \ and\ \bibinfo
  {author} {\bibfnamefont {D.~S.}\ \bibnamefont {Wishart}},\ }\bibfield
  {title} {\enquote {\bibinfo {title} {{SHIFTX2: Significantly improved protein
  chemical shift prediction}},}\ }\href {\doibase 10.1007/s10858-011-9478-4}
  {\bibfield  {journal} {\bibinfo  {journal} {Journal of Biomolecular NMR}\
  }\textbf {\bibinfo {volume} {50}},\ \bibinfo {pages} {43--57} (\bibinfo
  {year} {2011})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Zhang}, \citenamefont {Neal},\ and\ \citenamefont
  {Wishart}(2003)}]{Zhang2003}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {H.}~\bibnamefont
  {Zhang}}, \bibinfo {author} {\bibfnamefont {S.}~\bibnamefont {Neal}}, \ and\
  \bibinfo {author} {\bibfnamefont {D.~S.}\ \bibnamefont {Wishart}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{RefDB: A database of
  uniformly referenced protein chemical shifts}},}\ }\href {\doibase
  10.1023/A:1022836027055} {\bibfield  {journal} {\bibinfo  {journal} {Journal
  of Biomolecular NMR}\ }\textbf {\bibinfo {volume} {25}},\ \bibinfo {pages}
  {173--195} (\bibinfo {year} {2003})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Salvatier}, \citenamefont {Wiecki},\ and\
  \citenamefont {Fonnesbeck}(2016)}]{Salvatier2016}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {J.}~\bibnamefont
  {Salvatier}}, \bibinfo {author} {\bibfnamefont {T.~V.}\ \bibnamefont
  {Wiecki}}, \ and\ \bibinfo {author} {\bibfnamefont {C.}~\bibnamefont
  {Fonnesbeck}},\ }\bibfield  {title} {\enquote {\bibinfo {title}
  {{Probabilistic programming in Python using PyMC3}},}\ }\href {\doibase
  10.7717/peerj-cs.55} {\bibfield  {journal} {\bibinfo  {journal} {PeerJ
  Comput. Sci.}\ }\textbf {\bibinfo {volume} {2016}},\ \bibinfo {pages} {1--24}
  (\bibinfo {year} {2016})},\ \Eprint {http://arxiv.org/abs/1507.08050}
  {arXiv:1507.08050} \BibitemShut {NoStop}%
\bibitem [{\citenamefont {Kumar}\ \emph {et~al.}(2019)\citenamefont {Kumar},
  \citenamefont {Carroll}, \citenamefont {Hartikainen},\ and\ \citenamefont
  {Martin}}]{arviz_2019}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {R.}~\bibnamefont
  {Kumar}}, \bibinfo {author} {\bibfnamefont {C.}~\bibnamefont {Carroll}},
  \bibinfo {author} {\bibfnamefont {A.}~\bibnamefont {Hartikainen}}, \ and\
  \bibinfo {author} {\bibfnamefont {O.~A.}\ \bibnamefont {Martin}},\ }\bibfield
   {title} {\enquote {\bibinfo {title} {{{\{}ArviZ{\}} a unified library for
  exploratory analysis of {\{}Bayesian{\}} models in {\{}Python{\}}}},}\ }\href
  {\doibase 10.21105/joss.01143} {\bibfield  {journal} {\bibinfo  {journal} {J.
  Open Source Softw.}\ } (\bibinfo {year} {2019}),\
  10.21105/joss.01143}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Bonomi}\ \emph {et~al.}(2016)\citenamefont {Bonomi},
  \citenamefont {Camilloni}, \citenamefont {Cavalli},\ and\ \citenamefont
  {Vendruscolo}}]{Bonomi2016}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {M.}~\bibnamefont
  {Bonomi}}, \bibinfo {author} {\bibfnamefont {C.}~\bibnamefont {Camilloni}},
  \bibinfo {author} {\bibfnamefont {A.}~\bibnamefont {Cavalli}}, \ and\
  \bibinfo {author} {\bibfnamefont {M.}~\bibnamefont {Vendruscolo}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Metainference: A Bayesian
  inference method for heterogeneous systems}},}\ }\href {\doibase
  10.1126/sciadv.1501177} {\bibfield  {journal} {\bibinfo  {journal} {Science
  Advances}\ }\textbf {\bibinfo {volume} {2}},\ \bibinfo {pages} {1--9}
  (\bibinfo {year} {2016})},\ \Eprint {http://arxiv.org/abs/1509.05684}
  {arXiv:1509.05684} \BibitemShut {NoStop}%
\bibitem [{\citenamefont {Bratholm}\ and\ \citenamefont
  {Jensen}(2017)}]{Bratholm2017}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {L.~A.}\ \bibnamefont
  {Bratholm}}\ and\ \bibinfo {author} {\bibfnamefont {J.~H.}\ \bibnamefont
  {Jensen}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{Protein
  structure refinement using a quantum mechanics-based chemical shielding
  predictor}},}\ }\href {\doibase 10.1039/c6sc04344e} {\bibfield  {journal}
  {\bibinfo  {journal} {Chemical Science}\ }\textbf {\bibinfo {volume} {8}},\
  \bibinfo {pages} {2061--2072} (\bibinfo {year} {2017})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Bratholm}\ \emph {et~al.}(2015)\citenamefont
  {Bratholm}, \citenamefont {Christensen}, \citenamefont {Hamelryck},\ and\
  \citenamefont {Jensen}}]{Bratholm2015}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {L.~A.}\ \bibnamefont
  {Bratholm}}, \bibinfo {author} {\bibfnamefont {A.~S.}\ \bibnamefont
  {Christensen}}, \bibinfo {author} {\bibfnamefont {T.}~\bibnamefont
  {Hamelryck}}, \ and\ \bibinfo {author} {\bibfnamefont {J.~H.}\ \bibnamefont
  {Jensen}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{Bayesian
  inference of protein structure fromchemical shift data}},}\ }\href {\doibase
  10.7717/peerj.861} {\bibfield  {journal} {\bibinfo  {journal} {PeerJ}\
  }\textbf {\bibinfo {volume} {2015}},\ \bibinfo {pages} {1--19} (\bibinfo
  {year} {2015})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Sanjeevi}\ and\ \citenamefont
  {Viswanathan}(2017)}]{sanjeevi2017nuts}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {P.}~\bibnamefont
  {Sanjeevi}}\ and\ \bibinfo {author} {\bibfnamefont {P.}~\bibnamefont
  {Viswanathan}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{NUTS
  scheduling approach for cloud data centers to optimize energy
  consumption}},}\ }\href@noop {} {\bibfield  {journal} {\bibinfo  {journal}
  {Computing}\ }\textbf {\bibinfo {volume} {99}},\ \bibinfo {pages}
  {1179--1205} (\bibinfo {year} {2017})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Sawilowsky}(2009)}]{Sawilowsky2009}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {S.~S.}\ \bibnamefont
  {Sawilowsky}},\ }\bibfield  {title} {\enquote {\bibinfo {title} {{Very large
  and huge effect sizes}},}\ }\href {\doibase 10.22237/jmasm/1257035100}
  {\bibfield  {journal} {\bibinfo  {journal} {J. Mod. Appl. Stat. Methods}\
  }\textbf {\bibinfo {volume} {8}},\ \bibinfo {pages} {597--599} (\bibinfo
  {year} {2009})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {Kopec}, \citenamefont {Rothberg},\ and\ \citenamefont
  {de~Groot}(2019)}]{kopec2019molecular}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {W.}~\bibnamefont
  {Kopec}}, \bibinfo {author} {\bibfnamefont {B.~S.}\ \bibnamefont {Rothberg}},
  \ and\ \bibinfo {author} {\bibfnamefont {B.~L.}\ \bibnamefont {de~Groot}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Molecular mechanism of a
  potassium channel gating through activation gate-selectivity filter
  coupling}},}\ }\href@noop {} {\bibfield  {journal} {\bibinfo  {journal} {Nat.
  Commun.}\ }\textbf {\bibinfo {volume} {10}},\ \bibinfo {pages} {1--15}
  (\bibinfo {year} {2019})}\BibitemShut {NoStop}%
\bibitem [{\citenamefont {van~der Cruijsen}\ \emph {et~al.}(2017)\citenamefont
  {van~der Cruijsen}, \citenamefont {Prokofyev}, \citenamefont {Pongs},\ and\
  \citenamefont {Baldus}}]{VanderCruijsen2017}%
  \BibitemOpen
  \bibfield  {author} {\bibinfo {author} {\bibfnamefont {E.~A.}\ \bibnamefont
  {van~der Cruijsen}}, \bibinfo {author} {\bibfnamefont {A.~V.}\ \bibnamefont
  {Prokofyev}}, \bibinfo {author} {\bibfnamefont {O.}~\bibnamefont {Pongs}}, \
  and\ \bibinfo {author} {\bibfnamefont {M.}~\bibnamefont {Baldus}},\
  }\bibfield  {title} {\enquote {\bibinfo {title} {{Probing Conformational
  Changes during the Gating Cycle of a Potassium Channel in Lipid Bilayers}},}\
  }\href {\doibase 10.1016/j.bpj.2016.12.001} {\bibfield  {journal} {\bibinfo
  {journal} {Biophysical Journal}\ }\textbf {\bibinfo {volume} {112}},\
  \bibinfo {pages} {99--108} (\bibinfo {year} {2017})}\BibitemShut {NoStop}%
\end{thebibliography}%

\end{document}
%
% ****** End of file aipsamp.tex ******
